<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="SpectroPipeR">
<title>SpectroPipeR - step 1 - read Spectronaut data • SpectroPipeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="SpectroPipeR - step 1 - read Spectronaut data">
<meta property="og:description" content="SpectroPipeR">
<meta property="og:image" content="https://stemicha.github.io/SpectroPipeR/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light" data-bs-theme="light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">SpectroPipeR</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.3.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/a01_SpectroPipeR_read_Spectronaut_data.html">SpectroPipeR - step 1 - read Spectronaut data</a>
    <a class="dropdown-item" href="../articles/a02_SpectroPipeR_norm_quant.html">SpectroPipeR - step 2 - normalization and quantification</a>
    <a class="dropdown-item" href="../articles/a03_SpectroPipeR_MVA_analysis.html">SpectroPipeR - step 3 -  MVA analysis</a>
    <a class="dropdown-item" href="../articles/a04_SpectroPipeR_statistics.html">SpectroPipeR - step 4 - statistics</a>
    <a class="dropdown-item" href="../articles/a05_SpectroPipeR_reporting.html">SpectroPipeR - step 5 - reporting</a>
    <a class="dropdown-item" href="../articles/a06_SpectroPipeR_XIC_plot.html">SpectroPipeR - XIC plotting</a>
    <a class="dropdown-item" href="../articles/a07_Gprofiler2_code_suggestion.html">SpectroPipeR - stat. results functional enrichment analysis</a>
  </div>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav"></ul>
</div>

    
  </div>
</nav><div class="container template-article">



<script src="a01_SpectroPipeR_read_Spectronaut_data_files/kePrint-0.0.1/kePrint.js"></script><link href="a01_SpectroPipeR_read_Spectronaut_data_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>SpectroPipeR - step 1 - read Spectronaut data</h1>
            
      
      
      <div class="d-none name"><code>a01_SpectroPipeR_read_Spectronaut_data.Rmd</code></div>
    </div>

    
    
<div id="Tip1" class="greeting message" style="color: blue;">
<p><strong>If you want to have a closer look at the images and plots
just do a “right-click” on the image and select “open in new
tab”</strong></p>
</div>
<div id="Tip2" class="greeting message" style="color: green;">
<p><strong>The output folders contained figures that were always
exported in two file formats: vector PDF and raster PNG.</strong></p>
<p><u>Recommendations for MacOS Users</u></p>
<p>For MacOS users, it is recommended to use the PDF files, for example
in Keynote, to achieve better quality.</p>
<p><u>Recommendations for Windows Users</u></p>
<p>For Windows users, it is recommended to use the PNG files, as
applications like Microsoft PowerPoint can sometimes have issues when
working with vector graphics.</p>
</div>
<div class="section level2">
<h2 id="requirements">Requirements<a class="anchor" aria-label="anchor" href="#requirements"></a>
</h2>
<p>SpectroPipeR require the Quarto CLI. please make sure you installed
it on your system ! <a href="https://quarto.org/docs/get-started/" class="external-link">https://quarto.org/docs/get-started/</a></p>
<p>SpectroPipeR also requires certain columns from the Spectronaut
output report that are not included by default.</p>
<p><u>The following steps are advised:</u></p>
<ol style="list-style-type: decimal">
<li>Proceed with the download and installation of SpectroPipeR as
directed.</li>
<li>Utilize the <code><a href="../reference/Spectronaut_export_scheme.html">Spectronaut_export_scheme()</a></code> function to
create the necessary Spectronaut report scheme (*.rs).</li>
<li>Import the generated Spectronaut report scheme.</li>
<li>Conduct an analysis of your raw mass spectrometry data in
Spectronaut and produce the output report using the imported report
scheme.</li>
</ol>
<div class="section level3">
<h3 id="spectronaut-output-report-file">Spectronaut output report file<a class="anchor" aria-label="anchor" href="#spectronaut-output-report-file"></a>
</h3>
<p>Spectronaut output report should contain the following columns to
work in SpectroPipeR:</p>
<table class="table">
<thead><tr class="header">
<th align="right">mandatory Spectronaut columns</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="right">R.FileName</td>
</tr>
<tr class="even">
<td align="right">R.Condition</td>
</tr>
<tr class="odd">
<td align="right">R.Replicate</td>
</tr>
<tr class="even">
<td align="right">R.Run Date</td>
</tr>
<tr class="odd">
<td align="right">R.Instrument Name</td>
</tr>
<tr class="even">
<td align="right">R.Raw File Name</td>
</tr>
<tr class="odd">
<td align="right">R.MS1 Mass Analyzer</td>
</tr>
<tr class="even">
<td align="right">R.MS2 Mass Analyzer</td>
</tr>
<tr class="odd">
<td align="right">PG.ProteinGroups</td>
</tr>
<tr class="even">
<td align="right">PG.Organisms</td>
</tr>
<tr class="odd">
<td align="right">PG.IBAQ</td>
</tr>
<tr class="even">
<td align="right">PEP.StrippedSequence</td>
</tr>
<tr class="odd">
<td align="right">EG.ModifiedPeptide</td>
</tr>
<tr class="even">
<td align="right">PEP.NrOfMissedCleavages</td>
</tr>
<tr class="odd">
<td align="right">EG.UserGroup</td>
</tr>
<tr class="even">
<td align="right">EG.Qvalue</td>
</tr>
<tr class="odd">
<td align="right">EG.Cscore</td>
</tr>
<tr class="even">
<td align="right">EG.NormalizationFactor</td>
</tr>
<tr class="odd">
<td align="right">EG.TotalQuantity (Settings)</td>
</tr>
<tr class="even">
<td align="right">EG.SignalToNoise</td>
</tr>
<tr class="odd">
<td align="right">EG.Identified</td>
</tr>
<tr class="even">
<td align="right">EG.ApexRT</td>
</tr>
<tr class="odd">
<td align="right">EG.IntCorrScore</td>
</tr>
<tr class="even">
<td align="right">EG.DatapointsPerPeak</td>
</tr>
<tr class="odd">
<td align="right">EG.DatapointsPerPeak (MS1)</td>
</tr>
<tr class="even">
<td align="right">FG.Charge</td>
</tr>
<tr class="odd">
<td align="right">FG.Id</td>
</tr>
<tr class="even">
<td align="right">FG.XICDBID</td>
</tr>
<tr class="odd">
<td align="right">FG.LabeledSequence</td>
</tr>
<tr class="even">
<td align="right">FG.ShapeQualityScore</td>
</tr>
<tr class="odd">
<td align="right">FG.MS1Quantity</td>
</tr>
<tr class="even">
<td align="right">FG.MS2Quantity</td>
</tr>
<tr class="odd">
<td align="right">FG.MS1RawQuantity</td>
</tr>
<tr class="even">
<td align="right">FG.MS2RawQuantity</td>
</tr>
</tbody>
</table>
<p>The Spectronaut_export_scheme() function can be utilized to generate
a Spectronaut report template (*.rs). This template should then be
imported into Spectronaut. Subsequently, the analysis of your mass
spectrometry raw data should be conducted and documented using this
template within Spectronaut.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/Spectronaut_export_scheme.html">Spectronaut_export_scheme</a></span><span class="op">(</span>output_location <span class="op">=</span> <span class="st">"../SpectroPipeR_test_folder"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="parameters">parameters<a class="anchor" aria-label="anchor" href="#parameters"></a>
</h2>
<p>The global SpectroPipeR analysis paramaters is a list element
containing the basic analysis project information SpectroPipeR needs to
process the data.</p>
<table class="table">
<colgroup>
<col width="23%">
<col width="76%">
</colgroup>
<thead><tr class="header">
<th align="left"><u> <strong>parameter</strong> </u></th>
<th align="left"><u> <strong>description</strong> </u></th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">output_folder</td>
<td align="left">
<em>character</em> - output folder path (abs.)</td>
</tr>
<tr class="even">
<td align="left">ion_q_value_cutoff</td>
<td align="left">
<em>numeric</em> - Q-value used in Spectronaut
analysis: Biognosys default is 0.01 = 1% error rate</td>
</tr>
<tr class="odd">
<td align="left">id_drop_cutoff</td>
<td align="left">
<em>numeric</em> - value between 0-1 (1 = 100%); xx
percent lower than median of ion ID rate =&gt; outlier</td>
</tr>
<tr class="even">
<td align="left">normalization_method</td>
<td align="left">
<em>character</em> - “median” or Spectronaut;
auto-detection is per default ON, meaning if normalization was performed
in Spectronaut this will be detected and preferred over parameter
setting here; median normalization is the fallback option</td>
</tr>
<tr class="odd">
<td align="left">normalization_factor_cutoff_outlier</td>
<td align="left">
<em>numeric</em> - median off from global median (4
means abs. 4fold off)</td>
</tr>
<tr class="even">
<td align="left">filter_oxidized_peptides</td>
<td align="left">
<em>logical</em> - if oxidized peptides should be
removed from peptide quantification</td>
</tr>
<tr class="odd">
<td align="left">protein_intensity_estimation</td>
<td align="left">
<em>character</em> - Hi3 = Hi3 protein intensity
estimation, MaxLFQ = MaxLFQ protein intensity estimation</td>
</tr>
<tr class="even">
<td align="left">stat_test</td>
<td align="left">
<em>character</em> - choose statistical test: “rots” =
reproducibility optimized test statistics, “modt” = moderate t-test
(lmfit, eBayes), “t” = t-test</td>
</tr>
<tr class="odd">
<td align="left">type_slr</td>
<td align="left">
<em>character</em> - choose ratio aggregation method:
“median” or “tukey” is used when calculating protein values</td>
</tr>
<tr class="even">
<td align="left">fold_change</td>
<td align="left">
<em>numeric</em> - fold-change used as cutoff
e.g. 1.5</td>
</tr>
<tr class="odd">
<td align="left">p_value_cutoff</td>
<td align="left">
<em>numeric</em> - p-value used as cutoff
e.g. 0.05</td>
</tr>
<tr class="even">
<td align="left">paired</td>
<td align="left">
<em>logical</em> - Should paired statistics be
applied?</td>
</tr>
</tbody>
</table>
<div id="Attention" class="greeting message" style="color: red;">
<p><strong>Attention:</strong></p>
<p>If a Q-value of 0.01 is applied in Spectronaut and a Q-value of 0.001
is established within SpectroPipeR, it will impact the ID counting.
However, this change will NOT affect the quantification. The Spectronaut
report will be used as it is for quantification if no other filtering
was selected.</p>
</div>
<div class="section level3">
<h3 id="filter_oxidized_peptides-parameter">filter_oxidized_peptides parameter<a class="anchor" aria-label="anchor" href="#filter_oxidized_peptides-parameter"></a>
</h3>
<p>In SpectroPipeR, users can optionally set the
<code>filter_oxidized_peptides</code> parameter to <code>TRUE</code> to
remove peptides containing oxidized methionines while retaining their
unmodified counterparts. This filtering is recommended because
experiments using HYE species mixtures have demonstrated that
methionine-oxidized peptides exhibit significantly lower intensity and
suffer from markedly reduced relative quantitative precision compared to
their unmodified forms. Additionally, the variation between replicate
measurements is also substantially higher for oxidized peptides.
Consequently, removing methionine-oxidized peptides before
quantification and statistical analysis is the recommended approach in
SpectroPipeR to ensure optimal results.</p>
<div class="section level4">
<h4 id="hye-species-mix-experiements">HYE species mix experiements<a class="anchor" aria-label="anchor" href="#hye-species-mix-experiements"></a>
</h4>
<p>The HYE species mix experiments were carried out with 4 replicates as
described in <a href="https://www.nature.com/articles/nbt.3685" class="external-link">A
multicenter study benchmarks software tools for label-free proteome
quantification</a>. The data can be accessed via MassIVE (MSV000092489)
- <a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=0f33717d84fd45b1a318ad40670022cc" class="external-link uri">https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=0f33717d84fd45b1a318ad40670022cc</a>.</p>
<p>Spectronaut version 18.6 was used for data analysis. Following
filtering, the ion data only included peptides containing methionine
oxidation without prior carbamidomethylation, along with their
unmodified counterparts. To determine quantitative ratios, mix A ion
intensities were divided by the corresponding ion intensities from mix
B.</p>
<div class="float">
<img src="figures/HYE_mix_composition.png" style="width:30.0%" alt="HYE species mix composition"><div class="figcaption">HYE species mix composition</div>
</div>
<div class="section level5">
<h5 id="hye-species-mix-experiements-results-regarding-methionine-oxidation">HYE species mix experiements results regarding methionine
oxidation<a class="anchor" aria-label="anchor" href="#hye-species-mix-experiements-results-regarding-methionine-oxidation"></a>
</h5>
<div class="float">
<img src="figures/ox_vs_nonox__CV_plot.png" style="width:80.0%" alt="HYE species mix results: coefficient of variation plot of ions containing oxidized methionines and their unmodified forms"><div class="figcaption">HYE species mix results: coefficient of
variation plot of ions containing oxidized methionines and their
unmodified forms</div>
</div>
<div class="float">
<img src="figures/ox_vs_nonox__hist_plot.png" style="width:80.0%" alt="HYE species mix results: histogram of mean ion intensity of ions containing oxidized methionines and their unmodified forms"><div class="figcaption">HYE species mix results: histogram of mean ion
intensity of ions containing oxidized methionines and their unmodified
forms</div>
</div>
<div class="float">
<img src="figures/ox_vs_nonox__box_plot.png" style="width:80.0%" alt="HYE species mix results: ratios (A/B) boxplot of ions containing oxidized methionines and their unmodified forms"><div class="figcaption">HYE species mix results: ratios (A/B) boxplot of
ions containing oxidized methionines and their unmodified forms</div>
</div>
</div>
</div>
</div>
</div>
<div class="section level2">
<h2 id="spectropiper---output-folder-structure">SpectroPipeR - output folder structure<a class="anchor" aria-label="anchor" href="#spectropiper---output-folder-structure"></a>
</h2>
<p>The specified output folder is essential. If it does not exist it
will be created. Inside a SpectroPipeR specific folder structure will be
generated.</p>
<div class="float">
<img src="figures/folder_structure_OVERVIEW.png" style="width:50.0%" alt="SpectroPipeR output main folder structure"><div class="figcaption">SpectroPipeR output main folder structure</div>
</div>
</div>
<div class="section level2">
<h2 id="spectropiper---read-spectronaut-data-module">SpectroPipeR - read Spectronaut data module<a class="anchor" aria-label="anchor" href="#spectropiper---read-spectronaut-data-module"></a>
</h2>
<p>The <code><a href="../reference/read_spectronaut_module.html">read_spectronaut_module()</a></code> function serves to ingest
and process data from the Spectronaut software platform. This function
performs the following key tasks:</p>
<p><u><strong>data loading:</strong></u> - Reads in the raw data from
the Spectronaut report - Prepares the data for use within the
SpectroPipeR analysis environment</p>
<p><u><strong>ID analysis:</strong></u> - Provides feedback on key
qualitative characteristics of the analysis - Includes metrics such as:
- Identification (ID) rates - ON/OFF analysis results - Other relevant
qualitative insights</p>
<div class="section level3">
<h3 id="read_spectronaut_module-workflow">read_spectronaut_module() workflow<a class="anchor" aria-label="anchor" href="#read_spectronaut_module-workflow"></a>
</h3>
<ol style="list-style-type: decimal">
<li>check parameters</li>
<li>genrate output folders</li>
<li>init log file</li>
<li>load spectronaut report data</li>
<li>check columns</li>
<li>write input data to <code>01_input_data</code> folder in specified
output folder</li>
<li>perform raw file name capping for better visibility inside
plots</li>
<li>reformat Q-value (EG.Qvalue = mixed column (character and
numeric))</li>
<li>(optional) ID condition filtering; keep only data for downstream
analysis which passes condition-wise percentage filtering</li>
<li>generate and export intermediate SDRF file</li>
<li>export short summary of ID rates to log file</li>
<li>performing counting of min. sample percentage where an ion was
detected for user feedback in log</li>
<li>count ID rates (ions, peptides, protein groups) by filtering for the
specified Q-value in parameters list</li>
<li>get ID rates for 1 or at least 2 peptide hits</li>
<li>determine ion ID median to estimate ID outliers on that basis
depending on the parameters list value</li>
<li>generate ID tables and ID plots</li>
<li>The ON/OFF analysis is conducted by filtering the data to include
only proteins with a minimum of 2 peptide hits that are present in at
least 50% of the sample replicates, with proteins meeting this criteria
being classified as “ON” and those failing to do so being designated as
“OFF”.</li>
<li>generate ON/OFF analysis tables, plots and UpSet analysis</li>
<li>extract missed cleavages rates globally and on run level</li>
</ol>
<div class="section level4">
<h4 id="sdrf-file">SDRF file<a class="anchor" aria-label="anchor" href="#sdrf-file"></a>
</h4>
<p>“Public proteomics data often lack essential metadata, limiting its
potential. To address this, we present lesSDRF, a tool to simplify the
process of metadata annotation, thereby ensuring that data leave a
lasting, impactful legacy well beyond its initial publication.”
<em>Claeys, T. et al., 2023, Nat. Commun. 14, 6743</em></p>
<p>SpectroPipeR generates a intermediate SDRF tsv file, please use <a href="https://lessdrf.streamlit.app" class="external-link uri">https://lessdrf.streamlit.app</a> for finalizing file before
submitting to a public repository. The “source name” is generated by
using the R.Condition and R.Replicate column. Please carefully check
column names and edit and refine them according to <a href="https://github.com/bigbio/proteomics-sample-metadata/tree/master/sdrf-proteomics" class="external-link uri">https://github.com/bigbio/proteomics-sample-metadata/tree/master/sdrf-proteomics</a>.</p>
<p>For further information please read <a href="https://www.nature.com/articles/s41467-023-42543-5" class="external-link">lesSDRF is
more: maximizing the value of proteomics data through streamlined
metadata annotation</a></p>
</div>
<div class="section level4">
<h4 id="onoff-analysis-details">ON/OFF analysis details<a class="anchor" aria-label="anchor" href="#onoff-analysis-details"></a>
</h4>
<ol style="list-style-type: decimal">
<li>determine number of replicates per condition</li>
<li>filtering the ions for the used Q-value cutoff specified in the
parameters</li>
<li>filter for ProteinGroups with 2 or more peptides</li>
<li>count ProteinGroups per condition</li>
<li>calculate the in how many replicates the ProteinGroups was
identified with 2 or more peptides with a Q-value cutoff below the
specified one in the parameters</li>
<li>filter the list for ProteinGroups found only at least 50% of
replicates per condition</li>
<li>generate the wide output</li>
</ol>
</div>
</div>
<div class="section level3">
<h3 id="example-code">example code<a class="anchor" aria-label="anchor" href="#example-code"></a>
</h3>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># load SpectroPipeR</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">SpectroPipeR</span><span class="op">)</span></span>
<span><span class="co"># example input file, bundled with SpectroPipeR package</span></span>
<span><span class="va">example_file_path</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"SN_test_HYE_mix_file.tsv"</span>, package<span class="op">=</span><span class="st">"SpectroPipeR"</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># use default parameters list</span></span>
<span><span class="va">params</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>output_folder <span class="op">=</span> <span class="st">"../SpectroPipeR_test_folder"</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># step 1: load Spectronaut data module</span></span>
<span><span class="va">SpectroPipeR_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/read_spectronaut_module.html">read_spectronaut_module</a></span><span class="op">(</span>file <span class="op">=</span> <span class="va">example_file_path</span>,</span>
<span>                                      parameter <span class="op">=</span> <span class="va">params</span>,</span>
<span>                                      print.plot <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># checking parameters ...</span></span>
<span><span class="co"># #*****************************************</span></span>
<span><span class="co"># # READ SPECTRONAUT MODULE</span></span>
<span><span class="co"># #*****************************************</span></span>
<span><span class="co"># </span></span>
<span><span class="co"># loading data ...</span></span>
<span><span class="co"># write input data to output folder ...                                          </span></span>
<span><span class="co"># R.FileName capping ...                                                                                    </span></span>
<span><span class="co"># 02_ID_rate/8_sample_analysis ATTENTION !!! ---&gt; folder already exists - files will be replaced !!!</span></span>
<span><span class="co"># _________ data set loaded with ... _________</span></span>
<span><span class="co"># number of raw files = 8</span></span>
<span><span class="co"># number of conditions = 2</span></span>
<span><span class="co"># number of ions without filtering = 16654</span></span>
<span><span class="co"># number of peptides without filtering = 12761</span></span>
<span><span class="co"># number of Protein groups without filtering = 1503</span></span>
<span><span class="co"># count profiled values ...</span></span>
<span><span class="co"># performing counting of min. sample percentage where an ion was detected...</span></span>
<span><span class="co"># ion Q-value cutoff &lt; 0.01</span></span>
<span><span class="co"># 12.5 % (1/8) is the min. sample percentage where an ion was detected</span></span>
<span><span class="co"># performing ID rate filtering ...</span></span>
<span><span class="co"># performing protein count over replicates (more than or equal 2 peptides) ...</span></span>
<span><span class="co"># performing protein count (&lt;2 and more than or equal 2 peptides) ...</span></span>
<span><span class="co"># _________ ON/OFF analysis: _________                                                                </span></span>
<span><span class="co"># ... filter with Q-value 0.01 ...</span></span>
<span><span class="co"># ... filter for 2 peptides and min. present in 50% of replicates ...</span></span>
<span><span class="co"># ... ON/OFF analysis write outputs ...</span></span>
<span><span class="co"># _________ ID rate per sample: _________                                                                 </span></span>
<span><span class="co"># ions: median = 16205; min. = 15575; max. = 16541</span></span>
<span><span class="co"># modified peptides: median = 12850; min. = 12409; max. = 13072</span></span>
<span><span class="co"># stripped peptides: median = 12468.5; min. = 12042; max. = 12682</span></span>
<span><span class="co"># protein groups: median = 1485.5; min. = 1470; max. = 1499</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="read_spectronaut_module-outputs">read_spectronaut_module() outputs<a class="anchor" aria-label="anchor" href="#read_spectronaut_module-outputs"></a>
</h3>
<p>The output in your specified output folder of the
read_spectronaut_module() function should look like in this example:</p>
<p><img src="figures/folder_structure_read.png" style="width:50.0%"></p>
<!-- figures read Spectronaut data -->
<div class="section level4">
<h4 class="tabset tabset-pills" id="figures">figures<a class="anchor" aria-label="anchor" href="#figures"></a>
</h4>
<div class="section level5">
<h5 id="id_counts_plot">ID_counts_plot<a class="anchor" aria-label="anchor" href="#id_counts_plot"></a>
</h5>
<p>The bar chart labeled as <strong>ID_counts_plot</strong> illustrates
the number of identifications (IDs) discovered at the ion, peptide, and
protein group levels, corresponding to user specified Q-value. It is
advisable to select the same Q-value as defined in Spectronaut.</p>
<p><img src="figures/01__ID_counts_plot.png" style="width:40.0%"></p>
</div>
<div class="section level5">
<h5 id="id_counts_plot_ion_filter">ID_counts_plot_ion_filter<a class="anchor" aria-label="anchor" href="#id_counts_plot_ion_filter"></a>
</h5>
<p>The bar chart labeled as <strong>ID_counts_plot_ion_filter</strong>
depicts only the ions counts corresponding to user specified Q-value.
The dotted line in the graph represents the median count of ions, while
the solid line indicates the threshold criteria for defining an outlier
run. If the ‘id_drop_cutoff’ parameter is set to 0.3, this implies that
the cutoff criteria for identifying potential ID outliers is determined
by the formula <code>median of ion count * (1-0.3)</code>. A run with an
ion count below this line would be highlighted in orange and labeled as
an ID outlier.</p>
<p><img src="figures/01__ID_counts_plot_ion_filter.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="id_ion_counts_plot">ID_ion_counts_plot<a class="anchor" aria-label="anchor" href="#id_ion_counts_plot"></a>
</h5>
<p>The bar chart, titled <strong>ID_ion_counts_plot</strong>, presents
the ion identifications (IDs) in a stacked barchart format. It
differentiates between ions discovered below the specified Q-value
threshold (depicted in blue), those found above the threshold (shown in
grey), and the profiled ions (represented in red).</p>
<p><img src="figures/01__ID_ion_counts_plot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="missed_cleavages_sample_wise">missed_cleavages_sample_wise<a class="anchor" aria-label="anchor" href="#missed_cleavages_sample_wise"></a>
</h5>
<p>The bar chart, titled <strong>ID_ion_counts_plot</strong>, depicts
the number of missed cleavages per run. The text above the bar indicates
the missed cleavage percentage.</p>
<p><img src="figures/01__missed_cleavages_sample_wise.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="missed_cleavages_sample_wise_percentage">missed_cleavages_sample_wise_PERCENTAGE<a class="anchor" aria-label="anchor" href="#missed_cleavages_sample_wise_percentage"></a>
</h5>
<p>The bar chart, titled
<strong>missed_cleavages_sample_wise_PERCENTAGE</strong>, shows the
number of peptides with x-missed cleavages (y-axis left side) and the
percentage (y-axis right side) per run.</p>
<p><img src="figures/01__missed_cleavages_sample_wise_PERCENTAGE.png" style="width:30.0%"></p>
</div>
<div class="section level5">
<h5 id="missed_cleavages_global">missed_cleavages_global<a class="anchor" aria-label="anchor" href="#missed_cleavages_global"></a>
</h5>
<p>The bar chart, titled <strong>missed_cleavages_global</strong>,
depicts the missed cleavage rate for the whole project.</p>
<p><img src="figures/01__missed_cleavages_global.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="detected_proteingroups__upsetr__plot">Detected_ProteinGroups__UpSetR__plot<a class="anchor" aria-label="anchor" href="#detected_proteingroups__upsetr__plot"></a>
</h5>
<p>The UpSet plot (**Detected_ProteinGroups__UpSetR__plot**) illustrates
the outcome from the ON/OFF analysis. The ON/OFF analysis is conducted
by filtering the data to include only proteins with a minimum of 2
peptide hits that are present in at least 50% of the sample replicates,
with proteins meeting this criteria being classified as “ON” and those
failing to do so being designated as “OFF”.</p>
<p>If you want to know about the principle of an UpSet plot visit <a href="https://en.wikipedia.org/wiki/UpSet_plot" class="external-link uri">https://en.wikipedia.org/wiki/UpSet_plot</a></p>
<p><img src="figures/01__Detected_ProteinGroups__UpSetR__plot.png" style="width:50.0%"></p>
<!-- tables read Spectronaut data -->
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="tables">tables<a class="anchor" aria-label="anchor" href="#tables"></a>
</h4>
<div class="section level5">
<h5 id="file_list-csv">file_list.csv<a class="anchor" aria-label="anchor" href="#file_list-csv"></a>
</h5>
<p>The <strong>file_list.csv</strong> table contains 4 columns and gives
a brief overview of the files used in the project</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.FileName_raw</u> is the un-capped version of the raw file
name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.FileName_raw
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
20230222_Exp_SM_MethMS_manual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="id_counts-csv">ID_counts.csv<a class="anchor" aria-label="anchor" href="#id_counts-csv"></a>
</h5>
<p>The <strong>ID_counts.csv</strong> table summarizes the ID
counts.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>distinct_ions</u> is the number of distinct ions identified with
the user specified Q-value</li>
<li>
<u>distinct_modified_peptides</u> is the number of distinct peptide
sequences incl. modification identified with the user specified
Q-value</li>
<li>
<u>distinct_peptides</u> is the number of distinct stripped peptide
sequences identified with the user specified Q-value</li>
<li>
<u>distinct_proteins</u> is the number of distinct protein groups
identified with the user specified Q-value</li>
<li>
<u>ion_ID_outlier</u> yes / no indication if the run was marked as
an ID outlier</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
distinct_ions
</th>
<th style="text-align:right;">
distinct_modified_peptides
</th>
<th style="text-align:right;">
distinct_peptides
</th>
<th style="text-align:right;">
distinct_proteins
</th>
<th style="text-align:left;">
ion_ID_outlier
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
15992
</td>
<td style="text-align:right;">
12709
</td>
<td style="text-align:right;">
12337
</td>
<td style="text-align:right;">
1479
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
15775
</td>
<td style="text-align:right;">
12531
</td>
<td style="text-align:right;">
12159
</td>
<td style="text-align:right;">
1473
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
15715
</td>
<td style="text-align:right;">
12500
</td>
<td style="text-align:right;">
12131
</td>
<td style="text-align:right;">
1474
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
15575
</td>
<td style="text-align:right;">
12409
</td>
<td style="text-align:right;">
12042
</td>
<td style="text-align:right;">
1470
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
16418
</td>
<td style="text-align:right;">
12991
</td>
<td style="text-align:right;">
12600
</td>
<td style="text-align:right;">
1494
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
16490
</td>
<td style="text-align:right;">
13041
</td>
<td style="text-align:right;">
12649
</td>
<td style="text-align:right;">
1499
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
16541
</td>
<td style="text-align:right;">
13072
</td>
<td style="text-align:right;">
12682
</td>
<td style="text-align:right;">
1498
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
16479
</td>
<td style="text-align:right;">
13050
</td>
<td style="text-align:right;">
12660
</td>
<td style="text-align:right;">
1492
</td>
<td style="text-align:left;">
no
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="ion_id_counts_fractions-csv">ion_ID_counts_fractions.csv<a class="anchor" aria-label="anchor" href="#ion_id_counts_fractions-csv"></a>
</h5>
<p>The <strong>ion_ID_counts_fractions.csv</strong> table summarizes the
ID counts below or above the user specified Q-value and the count of
profiled ions.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>&lt;0.01</u> count of ions below a Q-value of 0.01</li>
<li>
<u>&gt;0.01</u> count of ions above a Q-value of 0.01</li>
<li>
<u>profiled</u> count of ions that were profiled</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
&lt;0.01
</th>
<th style="text-align:right;">
&gt;0.01
</th>
<th style="text-align:right;">
profiled
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
15992
</td>
<td style="text-align:right;">
335
</td>
<td style="text-align:right;">
327
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
15775
</td>
<td style="text-align:right;">
438
</td>
<td style="text-align:right;">
441
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
15715
</td>
<td style="text-align:right;">
463
</td>
<td style="text-align:right;">
476
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
15575
</td>
<td style="text-align:right;">
543
</td>
<td style="text-align:right;">
536
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
16418
</td>
<td style="text-align:right;">
109
</td>
<td style="text-align:right;">
127
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
16490
</td>
<td style="text-align:right;">
91
</td>
<td style="text-align:right;">
73
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
16541
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
16479
</td>
<td style="text-align:right;">
83
</td>
<td style="text-align:right;">
92
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="missed_cleavages_sample_wise_percentage-csv">missed_cleavages_sample_wise_PERCENTAGE.csv<a class="anchor" aria-label="anchor" href="#missed_cleavages_sample_wise_percentage-csv"></a>
</h5>
<p>The <strong>missed_cleavages_sample_wise_PERCENTAGE.csv</strong>
table summarizes the missed cleavages counts per run.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>MC_0</u> number of peptides with 0 missed cleavages</li>
<li>
<u>MC_1</u> number of peptides with 1 missed cleavages</li>
<li>
<u>MC_2</u> number of peptides with 2 missed cleavages</li>
<li>
<u>total_peptide_count</u> total number of peptides per run</li>
<li>
<u>MC_peptide_count</u> peptide count of missed cleavage
peptides</li>
<li>
<u>MC_percentage</u> the percentage of missed cleavages per run</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
MC_0
</th>
<th style="text-align:right;">
MC_1
</th>
<th style="text-align:right;">
MC_2
</th>
<th style="text-align:right;">
total_peptide_count
</th>
<th style="text-align:right;">
MC_peptide_count
</th>
<th style="text-align:right;">
MC_percentage
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
10485
</td>
<td style="text-align:right;">
1756
</td>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
12337
</td>
<td style="text-align:right;">
1852
</td>
<td style="text-align:right;">
15.01175
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:right;">
10341
</td>
<td style="text-align:right;">
1725
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
12159
</td>
<td style="text-align:right;">
1818
</td>
<td style="text-align:right;">
14.95189
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:right;">
10316
</td>
<td style="text-align:right;">
1721
</td>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
12131
</td>
<td style="text-align:right;">
1815
</td>
<td style="text-align:right;">
14.96167
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:right;">
10243
</td>
<td style="text-align:right;">
1707
</td>
<td style="text-align:right;">
92
</td>
<td style="text-align:right;">
12042
</td>
<td style="text-align:right;">
1799
</td>
<td style="text-align:right;">
14.93938
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:right;">
10727
</td>
<td style="text-align:right;">
1776
</td>
<td style="text-align:right;">
97
</td>
<td style="text-align:right;">
12600
</td>
<td style="text-align:right;">
1873
</td>
<td style="text-align:right;">
14.86508
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:right;">
10760
</td>
<td style="text-align:right;">
1791
</td>
<td style="text-align:right;">
98
</td>
<td style="text-align:right;">
12649
</td>
<td style="text-align:right;">
1889
</td>
<td style="text-align:right;">
14.93399
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:right;">
10785
</td>
<td style="text-align:right;">
1800
</td>
<td style="text-align:right;">
97
</td>
<td style="text-align:right;">
12682
</td>
<td style="text-align:right;">
1897
</td>
<td style="text-align:right;">
14.95821
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:right;">
10770
</td>
<td style="text-align:right;">
1792
</td>
<td style="text-align:right;">
98
</td>
<td style="text-align:right;">
12660
</td>
<td style="text-align:right;">
1890
</td>
<td style="text-align:right;">
14.92891
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="protein_count__strippedpep_without_qvalue_cut-csv">protein_count__strippedPEP_without_Qvalue_cut.csv<a class="anchor" aria-label="anchor" href="#protein_count__strippedpep_without_qvalue_cut-csv"></a>
</h5>
<p>The
<strong>protein_count__strippedPEP_without_Qvalue_cut.csv</strong> table
summarizes the protein group count without additional Q-value filtering
with at least 2 peptides or below 2 peptides.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>protein_count</u> protein group count</li>
<li>
<u>number_of_peptides</u> indicates if it is the fraction below 2
peptides per protein group (&lt;2) or the fraction of protein groups
that have at least 2 peptides (&gt;=2)</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
protein_count
</th>
<th style="text-align:left;">
number_of_peptides
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1334
</td>
<td style="text-align:left;">
&gt;= 2
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="protein_count__strippedpep-csv">protein_count__strippedPEP.csv<a class="anchor" aria-label="anchor" href="#protein_count__strippedpep-csv"></a>
</h5>
<p>The
<strong>protein_count__strippedPEP_without_Qvalue_cut.csv</strong> table
summarizes the protein group count with Q-value filtering
(e.g. &lt;0.01) with at least 2 peptides or below 2 peptides.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>protein_count</u> protein group count</li>
<li>
<u>number_of_peptides</u> indicates if it is the fraction below 2
peptides per protein group (&lt;2) or the fraction of protein groups
that have at least 2 peptides (&gt;=2)</li>
<li>
<u>Qvalue_below</u> user specififed Q-value threshold</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
protein_count
</th>
<th style="text-align:left;">
number_of_peptides
</th>
<th style="text-align:right;">
Qvalue_below
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
174
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
175
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
184
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
169
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
172
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
172
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
165
</td>
<td style="text-align:left;">
&lt; 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1310
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1299
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1299
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1286
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1325
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1327
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1326
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1327
</td>
<td style="text-align:left;">
&gt;= 2
</td>
<td style="text-align:right;">
0.01
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="detected_proteingroups__stripped_peptide_count_per_condition_raw_data-csv">Detected_ProteinGroups__stripped_peptide_count_per_condition_raw_data.csv<a class="anchor" aria-label="anchor" href="#detected_proteingroups__stripped_peptide_count_per_condition_raw_data-csv"></a>
</h5>
<p>The
<strong>Detected_ProteinGroups__stripped_peptide_count_per_condition_raw_data.csv</strong>
table summarizes the protein group count over replicates.</p>
<p>The table
<strong>protein_count_over_replicates_min_2_pep_Qvalue_cutoff_0.01.csv</strong>
is the same table without the peptide_count column.</p>
<p>The table
<strong>Detected_ProteinGroups__UpSetR__plot__stripped_peptide_count_per_condition.csv</strong>
summarize the same data in wide-tabular format containing the
information about the peptide count per condition</p>
<ul>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>peptide_count</u> number of stripped peptide sequencing after
filtering with the user specified Q-value</li>
<li>
<u>present_replicate_count</u> count in how many replicate the
protein group was found present</li>
<li>
<u>replicate_total_count</u> total replicate count of the
condition</li>
<li>
<u>present_replicate_percentage</u> percentage of in how many
replicate the protein group was found present</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
peptide_count
</th>
<th style="text-align:right;">
present_replicate_count
</th>
<th style="text-align:right;">
replicate_total_count
</th>
<th style="text-align:right;">
present_replicate_percentage
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O75643
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O95999
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P05719
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P08395
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0A6B4
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0A6K1
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0A742
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0A7B1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0A9G8
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0ACN4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0ACN7
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0ADR6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0ADX7
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0AEW4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0AFX9
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P0C0L9
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P16703
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P19097
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P23524
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P27240
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P27278
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P30863
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P31680
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P33607
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P37613
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P39371
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P52613
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P60757
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P66948
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P75780
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P75959
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P76113
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
P76231
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
Q01082
</td>
<td style="text-align:right;">
111
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
Q08554
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
Q5D862
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
O75643
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
P19097
</td>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
P22135
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
P32477
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
75
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
P33418
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
Q01082
</td>
<td style="text-align:right;">
111
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
Q12373
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
25
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="detected_proteingroups__stripped_peptide_count_per_condition_raw_data-xlsx">Detected_ProteinGroups__stripped_peptide_count_per_condition_raw_data.xlsx<a class="anchor" aria-label="anchor" href="#detected_proteingroups__stripped_peptide_count_per_condition_raw_data-xlsx"></a>
</h5>
<p>The table
<strong>Detected_ProteinGroups__stripped_peptide_count_per_condition_raw_data.xlsx</strong>
holds the binary information about the ON/OFF analysis results.</p>
<ul>
<li>
<u>PG.ProteinGroups </u> is protein group ID column</li>
<li>
<u>conditions…</u> the other columns highlighting in binary format
(1 = present / 0 = not present) if a protein group was found present
with the user specific criteria or not.</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
A_manual
</th>
<th style="text-align:right;">
B_manual
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14657
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14818
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14893
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14929
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14949
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O14981
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15014
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15061
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15091
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15160
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15212
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15355
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15400
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15460
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15504
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15511
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O15533
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O43172
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O43324
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
O43665
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1
</td>
</tr>
</tbody>
</table>
<p>The table
<strong>protein_count_over_replicates_min_2_pep_percentage_WIDE_Qvalue_cutoff_0.01.csv</strong>
summarize the data in the same way but here you can find the percentage
over replicates in a wide-tabular format.</p>
<p>The table
<strong>Detected_ProteinGroups__UpSetR__plot__binary_coded.csv</strong>
contains the binary information of the ON/OFF analysis for the UpSet
plot.</p>
</div>
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Stephan Michalik.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
