<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>SpectroPipeR - step 2 - normalization and quantification • SpectroPipeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.4.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="SpectroPipeR - step 2 - normalization and quantification">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">SpectroPipeR</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.3.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles">
<li><a class="dropdown-item" href="../articles/a01_SpectroPipeR_read_Spectronaut_data.html">SpectroPipeR - step 1 - read Spectronaut data</a></li>
    <li><a class="dropdown-item" href="../articles/a02_SpectroPipeR_norm_quant.html">SpectroPipeR - step 2 - normalization and quantification</a></li>
    <li><a class="dropdown-item" href="../articles/a03_SpectroPipeR_MVA_analysis.html">SpectroPipeR - step 3 -  MVA analysis</a></li>
    <li><a class="dropdown-item" href="../articles/a04_SpectroPipeR_statistics.html">SpectroPipeR - step 4 - statistics</a></li>
    <li><a class="dropdown-item" href="../articles/a05_SpectroPipeR_reporting.html">SpectroPipeR - step 5 - reporting</a></li>
    <li><a class="dropdown-item" href="../articles/a06_SpectroPipeR_XIC_plot.html">SpectroPipeR - additional feature - XIC plotting module</a></li>
    <li><a class="dropdown-item" href="../articles/a07_Gprofiler2_code_suggestion.html">SpectroPipeR - example code - functional enrichment analysis (external package)</a></li>
  </ul>
</li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">



<script src="a02_SpectroPipeR_norm_quant_files/kePrint-0.0.1/kePrint.js"></script><link href="a02_SpectroPipeR_norm_quant_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>SpectroPipeR - step 2 - normalization and quantification</h1>
            
      

      <div class="d-none name"><code>a02_SpectroPipeR_norm_quant.Rmd</code></div>
    </div>

    
    
<div id="Tip1" class="greeting message" style="color: blue;">
<p><strong>If you want to have a closer look at the images and plots
just do a “right-click” on the image and select “open in new
tab”</strong></p>
</div>
<div id="Tip2" class="greeting message" style="color: green;">
<p><strong>The output folders contained figures that were always
exported in two file formats: vector PDF and raster PNG.</strong></p>
<p><u>Recommendations for MacOS Users</u></p>
<p>For MacOS users, it is recommended to use the PDF files, for example
in Keynote, to achieve better quality.</p>
<p><u>Recommendations for Windows Users</u></p>
<p>For Windows users, it is recommended to use the PNG files, as
applications like Microsoft PowerPoint can sometimes have issues when
working with vector graphics.</p>
</div>
<div class="section level2">
<h2 id="spectropiper---normalization-and-quantification-module">SpectroPipeR - normalization and quantification module<a class="anchor" aria-label="anchor" href="#spectropiper---normalization-and-quantification-module"></a>
</h2>
<p>The <code><a href="../reference/norm_quant_module.html">norm_quant_module()</a></code> function serves to further
process the data from the <code><a href="../reference/read_spectronaut_module.html">read_spectronaut_module()</a></code>;
utilizes the read_spectronaut_module output:</p>
<p><u><strong>data normalization:</strong></u> - normalize the ion data
using the median-median normalization or auto-detect if the data was
already normalized in Spectronaut using e.g. local cross-run
normalization (If normalization was conducted using Spectronaut, the
median-median normalization step is omitted, and the data normalized by
Spectronaut is utilized.)</p>
<p><u><strong>(optional) batch adjusting:</strong></u> - provides the
batch adjusting functionality if needed using the ComBat methodology
described in <a href="https://academic.oup.com/biostatistics/article/8/1/118/252073" class="external-link">Johnson
et al. 2007</a></p>
<p><u><strong>(optional) covariate adjusting:</strong></u> - provides
the covariate adjusting functionality if needed using the lm() function
and the user specified meta data and formula to calculate the residuals
per peptides which are back-transformed to intensities by adding the
mean peptide intensity.</p>
<p><u><strong>protein quantification:</strong></u> - protein
quantification can be either done by Hi3 (project-wide determination of
Hi3 peptides) or MaxLFQ approach (<a href="https://academic.oup.com/bioinformatics/article/36/8/2611/5697917" class="external-link">iq
package</a>)</p>
<p>EG.TotalQuantity (Settings) column is used for the quantification.
Per default MS2 level should be selected in the quantification setting
in Spectronaut™.</p>
<div class="section level3">
<h3 id="norm_quant_module-workflow">norm_quant_module() workflow<a class="anchor" aria-label="anchor" href="#norm_quant_module-workflow"></a>
</h3>
<ol style="list-style-type: decimal">
<li>load read_spectronaut_module() output</li>
<li>(optional) check batch or covariate adjustment inputs</li>
<li>auto-detect if normalization was done inside Spectrout, if yes use
this normalization if not use median-median normalization</li>
<li>calculate/ectract normalization factors</li>
<li>generate plots for normalization factors/raw/normalized ion
intensities</li>
<li>calculate ion coefficient of variation (CV) globally and per missed
cleavage</li>
<li>calculate peptide intensities by summing up the ion intensities per
peptide sequence</li>
<li>replacing 0 peptide intensity values with the half-minimal peptide
intensity value</li>
<li>(optional) filter out methionine-oxidized peptides</li>
<li>(optional) perform batch adjusting using ComBat; PCA of data before
and after adjusting</li>
<li>(optional) perform covariate adjusting of data; PCA of data before
and after adjusting</li>
<li>calculate protein intensity from peptide intensities (Hi3 or
MaxLFQ)</li>
<li>extract iBAQ intensities (optional - covariate adjustment)</li>
<li>calculate protein intensity CV</li>
<li>compare protein int. (Hi3 or MaxLFQ) vs. iBAQ</li>
<li>calc. cumulative frequency of CV</li>
<li>generate sample to condition file including measurement order</li>
<li>generate plots &amp; table outputs</li>
</ol>
<div class="section level4">
<h4 id="background-informations">background informations<a class="anchor" aria-label="anchor" href="#background-informations"></a>
</h4>
<div class="section level5">
<h5 id="coefficient-of-variation-cv">coefficient of variation (CV)<a class="anchor" aria-label="anchor" href="#coefficient-of-variation-cv"></a>
</h5>
<p><code>coefficient of variation (CV) = standard deviation / mean</code></p>
<p>The coefficient of variation (CV) is a measure of the variability of
a dataset, and it is commonly used in proteomics to assess the
reproducibility of protein abundance measurements. In the given context,
the CV is observed to be higher in the low abundant range of the protein
intensity and lower in the mid to higher abundant range.</p>
<p>Several factors can influence the CV, including: sample preparation,
sample type, Mass Spectrometry (MS) methodology</p>
</div>
<div class="section level5">
<h5 id="hi3-protein-intensity">Hi3 protein intensity<a class="anchor" aria-label="anchor" href="#hi3-protein-intensity"></a>
</h5>
<p>Hi3 uses the mean over the highest 2-3 peptides per protein defined
by the median over the whole dataset.</p>
</div>
<div class="section level5">
<h5 id="ibaq-protein-intensity">iBAQ protein intensity<a class="anchor" aria-label="anchor" href="#ibaq-protein-intensity"></a>
</h5>
<p>The iBAQ (intensity-Based Absolute Quantification) is a method used
in proteomics to estimate the relative abundance of proteins within a
sample. The iBAQ value for a protein is calculated by dividing the total
intensity (sum of peptide intensities) of the protein by the number of
theoretically observable tryptic peptides for that protein. This
normalizes the protein intensity by the number of peptides that can be
detected. <a href="https://www.nature.com/articles/nature10098" class="external-link">Schwanhäusser *et
al., 2011</a></p>
</div>
<div class="section level5">
<h5 id="maxlfq-protein-intensity">MaxLFQ protein intensity<a class="anchor" aria-label="anchor" href="#maxlfq-protein-intensity"></a>
</h5>
<p>MaxLFQ stands for Maximal Peptide Ratio Extraction and Label-Free
Quantification. It is an algorithm used to estimate protein abundances
in mass spectrometry-based proteomics by aiming to maintain the fragment
intensity ratios between samples. The MaxLFQ algorithm calculates
protein intensities by taking the maximum peptide ratio of all peptides
that map to a protein and normalizing it across all samples.</p>
<p>The MaxLFQ algorithm was developed by <a href="https://doi.org/10.1074/mcp.M113.031591" class="external-link">Cox et al. in 2014</a>
and is widely used in label-free quantitative proteomics. It is
considered to be an accurate method for proteome-wide label-free
quantification.</p>
<p>In more technical terms, the MaxLFQ algorithm calculates ratio
between any two samples using the peptide species that are present. The
pair-wise protein ratio is then defined as the median of the peptide
ratios, to protect against outliers (require a minimal number of two
peptide ratios in order for a given protein ratio to be considered
valid). At this point the algorithm constructed a triangular matrix
containing all pair-wise protein ratios between any two samples, which
is the maximal possible quantification information. Then the algorithm
perform a least-squares analysis to reconstruct the abundance profile
optimally satisfying the individual protein ratios in the matrix based
on the sum of squared differences. Then the algorithm rescales the whole
profile to the cumulative intensity across samples, thereby preserving
the total summed intensity for a protein over all samples. This
procedure is repeated for all proteins, resulting in an accurate
abundance profile for each protein across the samples.</p>
</div>
<div class="section level5">
<h5 id="batch-adjustment-of-data-using-combat">Batch adjustment of data using ComBat<a class="anchor" aria-label="anchor" href="#batch-adjustment-of-data-using-combat"></a>
</h5>
<p>Batch effects refer to systematic differences between batches
(groups) of samples in high-throughput experiments. These differences
can arise due to various factors, such as batch variations in sample
preparation, handling, processing procedures and measurement orders.
Batch effects can obscure the true biological signal and lead to
incorrect conclusions if not properly accounted for. In the SpectroPipeR
pipeline, the ComBat tool was employed to adjust for batch effects in
the datasets where the batch covariate was known. ComBat utilizes the
methodology described in <a href="https://pubmed.ncbi.nlm.nih.gov/16632515/" class="external-link">Johnson et
al. 2007</a>. It uses an empirical Bayes (EB) framework for adjusting
data for batch effects that is robust to outliers in small sample sizes
and performs comparable to existing methods for large samples. <a href="https://pubmed.ncbi.nlm.nih.gov/16632515/" class="external-link">Johnson et
al. 2007:</a> This method incorporates systematic batch biases common
across genes in making adjustments, assuming that phenomena resulting in
batch effects often affect many genes in similar ways (i.e. increased
expression, higher variability, etc). Specifically, the L/S model
parameters are estimated that represent the batch effects by pooling
information across genes in each batch to shrink the batch effect
parameter estimates toward the overall mean of the batch effect
estimates (across peptides). These EB estimates are then used to adjust
the data for batch effects, providing more robust adjustments for the
batch effect on each peptide. In SpectroPipeR a parametric ComBAT
empirical Bayes adjustment is implemented by utilizing the sva-package.
After adjusting the data you may find a PCA analysis plot (Dim. 1-5) of
adjusted and un-adjusted data under <code>05_processed_data</code>.
Following the adjustment of peptide data, protein intensities are then
computed.</p>
</div>
<div class="section level5">
<h5 id="covariate-adjustment">Covariate adjustment<a class="anchor" aria-label="anchor" href="#covariate-adjustment"></a>
</h5>
<p>SpectroPipeR is capable of performing covariate adjustment on
quantitative data. This adjustment is achieved by utilizing
user-provided meta data and a formula. The adjustment process employs
the linear model function, lm(), and it operates on the log<sub>10</sub>
transformed peptide intensity data. Once the linear model is fitted, the
residuals are computed. These residuals are then adjusted back to the
original quantitative range by adding the mean peptide intensity across
all samples. This ensures that each peptide’s quantitative range is
preserved. Following the adjustment of peptide data, protein intensities
are then computed. Given that iBAQ intensities are derived from the
Spectronaut report, they undergo a similar adjustment process as was
applied to the peptides.</p>
<div class="float">
<img src="../articles/figures/Covariate_adj_example.png" style="width:100.0%" alt="covariate adjustment example on simulated peptide data (left pannel: peptide intensities; right pannel: age adjusted peptide intensities)"><div class="figcaption">covariate adjustment example on simulated
peptide data (left pannel: peptide intensities; right pannel: age
adjusted peptide intensities)</div>
</div>
</div>
</div>
</div>
<div class="section level3">
<h3 id="example-code">example code<a class="anchor" aria-label="anchor" href="#example-code"></a>
</h3>
<p><strong>norm_quant_module() needs the output of the
read_spectronaut_module() !</strong></p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># step 2: normalize &amp; quantification module</span></span>
<span><span class="va">SpectroPipeR_data_quant</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/norm_quant_module.html">norm_quant_module</a></span><span class="op">(</span>SpectroPipeR_data <span class="op">=</span> <span class="va">SpectroPipeR_data</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># #*****************************************</span></span>
<span><span class="co"># # NORMALIZATION &amp; QUANTIFICATION MODULE</span></span>
<span><span class="co"># #*****************************************</span></span>
<span><span class="co"># </span></span>
<span><span class="co"># sorting Replicates and conditions ...</span></span>
<span><span class="co"># NORMALIZATION WAS DONE IN SPECTRONAUT...</span></span>
<span><span class="co"># ...skipping normalization step and use Spectronaut normalized data instead...</span></span>
<span><span class="co"># save Normalization factor plot ...                                                                        </span></span>
<span><span class="co"># save Normalization boxplot ...</span></span>
<span><span class="co"># count missed cleavages ...</span></span>
<span><span class="co"># save missed cleavage plots...                                                                             </span></span>
<span><span class="co"># generate ion CV data...</span></span>
<span><span class="co"># save ion CV data plots...</span></span>
<span><span class="co"># save ion CV data vs. mean intensity hexbin plots...</span></span>
<span><span class="co"># calculating peptide intensity data ...</span></span>
<span><span class="co"># writing peptide intensity data ...</span></span>
<span><span class="co"># protein intensity calculation ...                                                                         </span></span>
<span><span class="co"># extracting iBAQ intensities from Spectronaut report ...</span></span>
<span><span class="co"># calc. mean, SD, CV of iBAQ intensities ...</span></span>
<span><span class="co"># ... save iBAQ data ...</span></span>
<span><span class="co"># perform maxLFQ protein intensity calculation ... (this will take some time)                               </span></span>
<span><span class="co"># ... preprocessing data for MaxLFQ estimation ...</span></span>
<span><span class="co"># Concatenating secondary ids...</span></span>
<span><span class="co"># </span></span>
<span><span class="co"># Removing low intensities...</span></span>
<span><span class="co"># </span></span>
<span><span class="co"># ... generate protein list for MaxLFQ estimation ...</span></span>
<span><span class="co"># # proteins = 1503, # samples = 8</span></span>
<span><span class="co"># 5.1%</span></span>
<span><span class="co"># 10%</span></span>
<span><span class="co"># 15%</span></span>
<span><span class="co"># 20%</span></span>
<span><span class="co"># 25%</span></span>
<span><span class="co"># 30%</span></span>
<span><span class="co"># 35%</span></span>
<span><span class="co"># 40%</span></span>
<span><span class="co"># 46%</span></span>
<span><span class="co"># 51%</span></span>
<span><span class="co"># 56%</span></span>
<span><span class="co"># 61%</span></span>
<span><span class="co"># 66%</span></span>
<span><span class="co"># 71%</span></span>
<span><span class="co"># 76%</span></span>
<span><span class="co"># 81%</span></span>
<span><span class="co"># 86%</span></span>
<span><span class="co"># 91%</span></span>
<span><span class="co"># 96%</span></span>
<span><span class="co"># Completed.</span></span>
<span><span class="co"># ... calculation of MaxLFQ ...</span></span>
<span><span class="co"># 5.1%</span></span>
<span><span class="co"># 10%</span></span>
<span><span class="co"># 15%</span></span>
<span><span class="co"># 20%</span></span>
<span><span class="co"># 25%</span></span>
<span><span class="co"># 30%</span></span>
<span><span class="co"># 35%</span></span>
<span><span class="co"># 40%</span></span>
<span><span class="co"># 46%</span></span>
<span><span class="co"># 51%</span></span>
<span><span class="co"># 56%</span></span>
<span><span class="co"># 61%</span></span>
<span><span class="co"># 66%</span></span>
<span><span class="co"># 71%</span></span>
<span><span class="co"># 76%</span></span>
<span><span class="co"># 81%</span></span>
<span><span class="co"># 86%</span></span>
<span><span class="co"># 91%</span></span>
<span><span class="co"># 96%</span></span>
<span><span class="co"># Completed.</span></span>
<span><span class="co"># ... generate outputs for MaxLFQ estimation ...</span></span>
<span><span class="co"># ... do median normalization of maxLFQ data ...</span></span>
<span><span class="co"># ... save MaxLFQ boxplot ...</span></span>
<span><span class="co"># ... save MaxLFQ data ...</span></span>
<span><span class="co"># ... compare protein intensities and iBAQ protein intensities ...                                           </span></span>
<span><span class="co"># ... CV plot calculation ...</span></span>
<span><span class="co"># ... render CV plot ...</span></span>
<span><span class="co"># _________ normalization done _________</span></span>
<span><span class="co"># no outlier detected with 4 fold difference from the median</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="norm_quant_module-outputs">norm_quant_module() outputs<a class="anchor" aria-label="anchor" href="#norm_quant_module-outputs"></a>
</h3>
<p>The output in your specified output folder for the
norm_quant_module() function should look like in this example
(03_normalization, 05_processed_data):</p>
<p><img src="../articles/figures/folder_structure_normalize.png" style="width:30.0%"></p>
<p><img src="../articles/figures/folder_structure_processed.png" style="width:50.0%"></p>
<!-- figures norm quant. Spectronaut data -->
<div class="section level4">
<h4 class="tabset tabset-pills" id="normalization---figures">normalization - figures<a class="anchor" aria-label="anchor" href="#normalization---figures"></a>
</h4>
<div class="section level5">
<h5 id="normalization_factor_plot">normalization_factor_plot<a class="anchor" aria-label="anchor" href="#normalization_factor_plot"></a>
</h5>
<p>The bar chart, denoted as <strong>normalization_factor_plot</strong>,
depicts the normalization factor employed for data normalization. In the
event that local cross-run normalization was chosen in Spectronaut, the
median of the normalization factors is exhibited. The user-defined
cut-off threshold in the SpectroPipeR parameters setting is represented
by the solid lines. If a run were to have a normalization factor
exceeding the threshold, it would be highlighted in orange on the plot
and indicated in the normalization tables.</p>
<p><img src="../articles/figures/02__normalization_factor_plot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="normalization_factor_boxplot">normalization_factor_BOXplot<a class="anchor" aria-label="anchor" href="#normalization_factor_boxplot"></a>
</h5>
<p>The boxplot/density chart, denoted as
<strong>normalization_factor_BOXplot</strong>, depicts the normalization
factor employed for data normalization e.g. if local cross-run
normalization was chosen in Spectronaut.</p>
<p><img src="../articles/figures/02__normalization_factor_BOXplot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="normalization_boxplot">normalization_boxplot<a class="anchor" aria-label="anchor" href="#normalization_boxplot"></a>
</h5>
<p>The boxplot/density chart, denoted as
<strong>normalization_boxplot</strong>, depicts the raw and normalized
ion intensities.</p>
<p><img src="../articles/figures/02__normalization_boxplot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="ion_cv_plot">ion_CV_plot<a class="anchor" aria-label="anchor" href="#ion_cv_plot"></a>
</h5>
<p>The <strong>ion_CV_plot</strong> illustrates the ion coefficient of
variation (CV) globally and assigned to missed cleavages.</p>
<p><img src="../articles/figures/02__ion_CV_plot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="ion_intenisty_vs_cv_hexbin_plot">ion_intenisty_vs_CV_hexbin_plot<a class="anchor" aria-label="anchor" href="#ion_intenisty_vs_cv_hexbin_plot"></a>
</h5>
<p>The <strong>ion_intenisty_vs_CV_hexbin_plot</strong> depicts the ion
coefficient of variation (CV) in regard to the normalized ion
intensity.</p>
<p><img src="../articles/figures/02__ion_intenisty_vs_CV_hexbin_plot.png" style="width:80.0%"></p>
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="normalization---tables">normalization - tables<a class="anchor" aria-label="anchor" href="#normalization---tables"></a>
</h4>
<div class="section level5">
<h5 id="median_normalization_factors-csv">Median_normalization_factors.csv<a class="anchor" aria-label="anchor" href="#median_normalization_factors-csv"></a>
</h5>
<p>The <strong>file_list.csv</strong> table contains 4 columns and gives
a brief overview of the files used in the project</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>MedianNormalizationFactor</u> is the normalization factor or in
case of local cross run normalization the median of the normalization
factors</li>
<li>
<u>normalization_outlier</u> indicator column for an outlier based
on the normalization factor threshold the user specified in the
SpectroPipeR parameters</li>
</ul>
<table class="table table" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
MedianNormalizationFactor
</th>
<th style="text-align:left;">
normalization_outlier
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.8117018
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.8680521
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.9313351
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.0809348
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.0050862
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1.0473045
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1.0161635
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
1.1874938
</td>
<td style="text-align:left;">
no
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="processed-data---figures">processed data - figures<a class="anchor" aria-label="anchor" href="#processed-data---figures"></a>
</h4>
<div class="section level5">
<h5 id="maxlfq_protein_intensity_boxplot">MaxLFQ_protein_intensity_boxplot<a class="anchor" aria-label="anchor" href="#maxlfq_protein_intensity_boxplot"></a>
</h5>
<p>The <strong>MaxLFQ_protein_intensity_boxplot</strong> illustrates the
raw and normalized MaxLFQ protein intensities.</p>
<p><img src="../articles/figures/02a__MaxLFQ_protein_intensity_boxplot.png" style="width:50.0%"></p>
</div>
<div class="section level5">
<h5 id="cv_vs_intensity_plot">CV_vs_intensity_plot<a class="anchor" aria-label="anchor" href="#cv_vs_intensity_plot"></a>
</h5>
<p>The <strong>CV_vs_intensity_plot</strong> depicts the normalized
protein intensity vs the coefficient of variation (CV) of the protein
intensity.</p>
<p>The horizontal solid line in the figure indicates a CV of 0.1, while
the dotted line represents a CV of 0.2. These lines serve as reference
points to evaluate the variability of the protein abundance
measurements.</p>
<p>The pink labels in the figure show the percentage of proteins that
have a CV below 0.1 or 0.2, respectively, in relation to the total
number of protein identifications. This information provides insights
into the overall reproducibility of the protein abundance measurements
within the dataset.</p>
<p><img src="../articles/figures/02a__CV_vs_intensity_plot.png" style="width:80.0%"></p>
</div>
<div class="section level5">
<h5 id="protein_intensities_vs_ibaq_intensities">protein_intensities_vs_iBAQ_intensities<a class="anchor" aria-label="anchor" href="#protein_intensities_vs_ibaq_intensities"></a>
</h5>
<p>The <strong>protein_intensities_vs_iBAQ_intensities</strong> plot
depicts the the ratios between different protein intensity estimation
algorithms, such as MaxLFQ and iBAQ. This allows the user to assess the
relationship and potential differences between these estimation
methods.</p>
<p>In the upper panel the plot displays the ratios between the protein
intensity estimates, providing a visual representation of the
similarities or discrepancies between the different algorithms.</p>
<p>The lower panel of the plot includes a bar chart that shows the count
or frequency of the individual protein intensity estimations.</p>
<p><img src="../articles/figures/02a__protein_intensities_vs_iBAQ_intensities.png" style="width:70.0%"></p>
</div>
<div class="section level5">
<h5 id="cv_cumulative_frequency_plot">CV_cumulative_frequency_plot<a class="anchor" aria-label="anchor" href="#cv_cumulative_frequency_plot"></a>
</h5>
<p>The <strong>CV_cumulative_frequency_plot</strong> graphically
represents the cumulative frequency of the Coefficient of Variation (CV)
at both the peptide and protein levels. On the x-axis, the coefficient
of variation (CV) is plotted, while the y-axis displays the cumulative
frequency. The lines are differentiated by color according to the
condition. This enables the user to assess and evaluate the
reproducibility of measurements across different conditions in the
analysis.</p>
<p><img src="../articles/figures/02a__CV_cumulative_frequency_plot.png" style="width:70.0%"></p>
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="processed-data---tables">processed data - tables<a class="anchor" aria-label="anchor" href="#processed-data---tables"></a>
</h4>
<div class="section level5">
<h5 id="sample_to_condition_file-csv">sample_to_condition_file.csv<a class="anchor" aria-label="anchor" href="#sample_to_condition_file-csv"></a>
</h5>
<p>The <strong>sample_to_condition_file.csv</strong> table contains the
information about the run file name, condition, replicate, run date and
resulting measurement order.</p>
<ul>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Run Date</u> raw file run date</li>
<li>
<u>measurement_order</u> measurement order (integer)</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:left;">
R.Run Date
</th>
<th style="text-align:right;">
measurement_order
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
2023-02-23 02:40:34
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
2023-02-23 04:48:36
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
2023-02-23 06:56:38
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
2023-02-23 09:04:40
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
2023-02-23 11:12:43
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
2023-02-23 13:20:47
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
2023-02-23 15:28:46
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
2023-02-23 17:36:48
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="peptide_intensities-csv">peptide_intensities.csv<a class="anchor" aria-label="anchor" href="#peptide_intensities-csv"></a>
</h5>
<p>The <strong>peptide_intensities.csv</strong> table contains the
information about the calculated peptide intensities. Specifically, this
table holds the sum of the normalized ion data for each peptide-sample
combination.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>EG.ModifiedPeptide</u> modified peptide sequences</li>
<li>
<u>PEP.StrippedSequence</u> stripped peptide sequences</li>
<li>
<u>peptide_intensity</u> peptide intensity</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
EG.ModifiedPeptide
</th>
<th style="text-align:left;">
PEP.StrippedSequence
</th>
<th style="text-align:right;">
peptide_intensity
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
<em>GEVPAMLPLK</em>
</td>
<td style="text-align:left;">
GEVPAMLPLK
</td>
<td style="text-align:right;">
12263.129
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
<em>LFDNTVGAYR</em>
</td>
<td style="text-align:left;">
LFDNTVGAYR
</td>
<td style="text-align:right;">
18185.355
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>AAAASVPNADGLK</em>
</td>
<td style="text-align:left;">
AAAASVPNADGLK
</td>
<td style="text-align:right;">
12529.687
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>EGWAPATFIDK</em>
</td>
<td style="text-align:left;">
EGWAPATFIDK
</td>
<td style="text-align:right;">
6625.379
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>FEGRPVPDGDAK</em>
</td>
<td style="text-align:left;">
FEGRPVPDGDAK
</td>
<td style="text-align:right;">
3225.517
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>LGEAAALENNTGSEATGPSRPLPDAPHGVMDSGLPWSK</em>
</td>
<td style="text-align:left;">
LGEAAALENNTGSEATGPSRPLPDAPHGVMDSGLPWSK
</td>
<td style="text-align:right;">
5767.410
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>SLLDGEGPQAVGGQDVAFSR</em>
</td>
<td style="text-align:left;">
SLLDGEGPQAVGGQDVAFSR
</td>
<td style="text-align:right;">
2322.913
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>SVPVPLQEAPQQR</em>
</td>
<td style="text-align:left;">
SVPVPLQEAPQQR
</td>
<td style="text-align:right;">
6064.211
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>TDLPEEKPDATPQNPFLK</em>
</td>
<td style="text-align:left;">
TDLPEEKPDATPQNPFLK
</td>
<td style="text-align:right;">
9141.116
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>VTWSSGSTEAIYR</em>
</td>
<td style="text-align:left;">
VTWSSGSTEAIYR
</td>
<td style="text-align:right;">
7835.685
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>YTVIYPYTAR</em>
</td>
<td style="text-align:left;">
YTVIYPYTAR
</td>
<td style="text-align:right;">
11841.916
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:left;">
<em>LTGNPELSSLDEVLAK</em>
</td>
<td style="text-align:left;">
LTGNPELSSLDEVLAK
</td>
<td style="text-align:right;">
248208.404
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>DYLQLLR</em>
</td>
<td style="text-align:left;">
DYLQLLR
</td>
<td style="text-align:right;">
55120.242
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>GFLAGYVVAK</em>
</td>
<td style="text-align:left;">
GFLAGYVVAK
</td>
<td style="text-align:right;">
40748.910
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>NQLESLQR</em>
</td>
<td style="text-align:left;">
NQLESLQR
</td>
<td style="text-align:right;">
37823.902
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="peptide_intensities_final-csv">peptide_intensities_final.csv<a class="anchor" aria-label="anchor" href="#peptide_intensities_final-csv"></a>
</h5>
<p>The <strong>peptide_intensities_final.csv</strong> table contains the
information about the calculated peptide intensities finalized by
e.g. removing methionine oxidized peptides.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>EG.ModifiedPeptide</u> modified peptide sequences</li>
<li>
<u>PEP.StrippedSequence</u> stripped peptide sequences</li>
<li>
<u>peptide_intensity</u> peptide intensity</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
EG.ModifiedPeptide
</th>
<th style="text-align:left;">
PEP.StrippedSequence
</th>
<th style="text-align:right;">
peptide_intensity
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
<em>GEVPAMLPLK</em>
</td>
<td style="text-align:left;">
GEVPAMLPLK
</td>
<td style="text-align:right;">
12263.129
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
<em>LFDNTVGAYR</em>
</td>
<td style="text-align:left;">
LFDNTVGAYR
</td>
<td style="text-align:right;">
18185.355
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>AAAASVPNADGLK</em>
</td>
<td style="text-align:left;">
AAAASVPNADGLK
</td>
<td style="text-align:right;">
12529.687
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>EGWAPATFIDK</em>
</td>
<td style="text-align:left;">
EGWAPATFIDK
</td>
<td style="text-align:right;">
6625.379
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>FEGRPVPDGDAK</em>
</td>
<td style="text-align:left;">
FEGRPVPDGDAK
</td>
<td style="text-align:right;">
3225.517
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>LGEAAALENNTGSEATGPSRPLPDAPHGVMDSGLPWSK</em>
</td>
<td style="text-align:left;">
LGEAAALENNTGSEATGPSRPLPDAPHGVMDSGLPWSK
</td>
<td style="text-align:right;">
5767.410
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>SLLDGEGPQAVGGQDVAFSR</em>
</td>
<td style="text-align:left;">
SLLDGEGPQAVGGQDVAFSR
</td>
<td style="text-align:right;">
2322.913
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>SVPVPLQEAPQQR</em>
</td>
<td style="text-align:left;">
SVPVPLQEAPQQR
</td>
<td style="text-align:right;">
6064.211
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>TDLPEEKPDATPQNPFLK</em>
</td>
<td style="text-align:left;">
TDLPEEKPDATPQNPFLK
</td>
<td style="text-align:right;">
9141.116
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>VTWSSGSTEAIYR</em>
</td>
<td style="text-align:left;">
VTWSSGSTEAIYR
</td>
<td style="text-align:right;">
7835.685
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
<em>YTVIYPYTAR</em>
</td>
<td style="text-align:left;">
YTVIYPYTAR
</td>
<td style="text-align:right;">
11841.916
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:left;">
<em>LTGNPELSSLDEVLAK</em>
</td>
<td style="text-align:left;">
LTGNPELSSLDEVLAK
</td>
<td style="text-align:right;">
248208.404
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>DYLQLLR</em>
</td>
<td style="text-align:left;">
DYLQLLR
</td>
<td style="text-align:right;">
55120.242
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>GFLAGYVVAK</em>
</td>
<td style="text-align:left;">
GFLAGYVVAK
</td>
<td style="text-align:right;">
40748.910
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
<em>NQLESLQR</em>
</td>
<td style="text-align:left;">
NQLESLQR
</td>
<td style="text-align:right;">
37823.902
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="ibaq_protein_intensity_data_extracted_from_spectronaut-csv">iBAQ_protein_intensity_data_extracted_from_Spectronaut.csv<a class="anchor" aria-label="anchor" href="#ibaq_protein_intensity_data_extracted_from_spectronaut-csv"></a>
</h5>
<p>The
<strong>iBAQ_protein_intensity_data_extracted_from_Spectronaut.csv</strong>
table contains the information about the calculated peptide intensities
finalized by e.g. removing methionine oxidized peptides.</p>
<ul>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>iBAQ_intensities</u> SpectroPipeR extracted iBAQ intensities</li>
<li>
<u>PG.IBAQ_raw</u> Spectronaut iBAQ calculations</li>
</ul>
<p>Since a protein group can contain more than one protein the number of
theoretical peptides may differ. Therefore Spectronaut separates with a
“;” e.g. “6179.5;6376.5” (iBAQ intensities from Spectronaut =
PG.IBAQ_raw). SpectroPipeR uses the mean of the iBAQ values to give an
iBAQ estimate for the protein group as well (iBAQ_intensities).</p>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
iBAQ_intensities
</th>
<th style="text-align:left;">
PG.IBAQ_raw
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
2537.4
</td>
<td style="text-align:left;">
2537.4
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
1361.5
</td>
<td style="text-align:left;">
1361.5
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:right;">
82736.1
</td>
<td style="text-align:left;">
82736.1
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
33003.4
</td>
<td style="text-align:left;">
33003.4
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
L0R8F8
</td>
<td style="text-align:right;">
2669.7
</td>
<td style="text-align:left;">
2669.7
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
4721.1
</td>
<td style="text-align:left;">
4721.1
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O00458
</td>
<td style="text-align:right;">
121.3
</td>
<td style="text-align:left;">
121.3
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
31489.5
</td>
<td style="text-align:left;">
31489.5
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
117313.4
</td>
<td style="text-align:left;">
117313.4
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
3771.2
</td>
<td style="text-align:left;">
3771.2
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
277582.3
</td>
<td style="text-align:left;">
277582.3
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O13547
</td>
<td style="text-align:right;">
1905.0
</td>
<td style="text-align:left;">
1905
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O14521
</td>
<td style="text-align:right;">
6462.5
</td>
<td style="text-align:left;">
6462.5
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
26804.5
</td>
<td style="text-align:left;">
26804.5
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
43005.4
</td>
<td style="text-align:left;">
43005.4
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
P21127;Q9UQ88
</td>
<td style="text-align:right;">
6179.5
</td>
<td style="text-align:left;">
6179.5;6179.5
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="ibaq_protein_intensity_data_extracted_from_spectronaut_summary-csv">iBAQ_protein_intensity_data_extracted_from_Spectronaut_summary.csv<a class="anchor" aria-label="anchor" href="#ibaq_protein_intensity_data_extracted_from_spectronaut_summary-csv"></a>
</h5>
<p>The
<strong>iBAQ_protein_intensity_data_extracted_from_Spectronaut_summary.csv</strong>
table contains the information about the calculated peptide intensities
finalized by e.g. removing methionine oxidized peptides.</p>
<ul>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>mean_iBAQ_intensities</u> mean iBAQ intensities over
replicates</li>
<li>
<u>SD_iBAQ_intensities</u> standard deviation of iBAQ intensities
over replicates</li>
<li>
<u>CV_iBAQ_intensities</u> coefficient of variation of iBAQ
intensities over replicates</li>
<li>
<u>iBAQ_quantiles</u> project specific iBAQ quantile calculated by
using the analysis specific mean of iBAQ intensities over all runs</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
mean_iBAQ_intensities
</th>
<th style="text-align:right;">
SD_iBAQ_intensities
</th>
<th style="text-align:right;">
CV_iBAQ_intensities
</th>
<th style="text-align:left;">
iBAQ_quantiles
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
2660.450
</td>
<td style="text-align:right;">
204.97937
</td>
<td style="text-align:right;">
0.0770469
</td>
<td style="text-align:left;">
Q5
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
1324.475
</td>
<td style="text-align:right;">
59.81551
</td>
<td style="text-align:right;">
0.0451617
</td>
<td style="text-align:left;">
Q4
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:right;">
78088.275
</td>
<td style="text-align:right;">
3575.86724
</td>
<td style="text-align:right;">
0.0457926
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
31907.450
</td>
<td style="text-align:right;">
1404.82917
</td>
<td style="text-align:right;">
0.0440282
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
L0R8F8
</td>
<td style="text-align:right;">
2966.800
</td>
<td style="text-align:right;">
633.34122
</td>
<td style="text-align:right;">
0.2134762
</td>
<td style="text-align:left;">
Q5
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
4884.950
</td>
<td style="text-align:right;">
228.75335
</td>
<td style="text-align:right;">
0.0468282
</td>
<td style="text-align:left;">
Q6
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O00458
</td>
<td style="text-align:right;">
150.875
</td>
<td style="text-align:right;">
19.88540
</td>
<td style="text-align:right;">
0.1318005
</td>
<td style="text-align:left;">
Q1
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
32368.550
</td>
<td style="text-align:right;">
741.58758
</td>
<td style="text-align:right;">
0.0229107
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
112171.575
</td>
<td style="text-align:right;">
6540.49731
</td>
<td style="text-align:right;">
0.0583080
</td>
<td style="text-align:left;">
Q10
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
3548.625
</td>
<td style="text-align:right;">
182.64097
</td>
<td style="text-align:right;">
0.0514681
</td>
<td style="text-align:left;">
Q5
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
251255.500
</td>
<td style="text-align:right;">
17827.74390
</td>
<td style="text-align:right;">
0.0709546
</td>
<td style="text-align:left;">
Q10
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O13547
</td>
<td style="text-align:right;">
1873.825
</td>
<td style="text-align:right;">
187.98415
</td>
<td style="text-align:right;">
0.1003211
</td>
<td style="text-align:left;">
Q4
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O14521
</td>
<td style="text-align:right;">
6567.325
</td>
<td style="text-align:right;">
183.61005
</td>
<td style="text-align:right;">
0.0279581
</td>
<td style="text-align:left;">
Q6
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
26587.425
</td>
<td style="text-align:right;">
707.79358
</td>
<td style="text-align:right;">
0.0266214
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
43786.775
</td>
<td style="text-align:right;">
1077.65641
</td>
<td style="text-align:right;">
0.0246115
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="maxlfq_protein_intensity_data-csv">maxLFQ_protein_intensity_data.csv<a class="anchor" aria-label="anchor" href="#maxlfq_protein_intensity_data-csv"></a>
</h5>
<p>The <strong>maxLFQ_protein_intensity_data.csv</strong> table contains
the information about the calculated MaxLFQ protein intensity data
calculated by the <a href="https://academic.oup.com/bioinformatics/article/36/8/2611/5697917" class="external-link">iq
package</a> algorithm.</p>
<ul>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>R.FileName</u> is the capped raw file name</li>
<li>
<u>protein_intensity</u> MaxLFQ intensity</li>
<li>
<u>R.Condition</u> is the condition naming which was setup in your
Spectronaut analysis</li>
<li>
<u>R.Replicate</u> is the replicate number which was setup in your
Spectronaut analysis</li>
<li>
<u>intensity_rank</u> MaxLFQ protein intensity rank</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
protein_intensity
</th>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
R.Replicate
</th>
<th style="text-align:right;">
intensity_rank
</th>
<th style="text-align:right;">
raw_protein_intensity
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
16246.423
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
497
</td>
<td style="text-align:right;">
14933.498
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:right;">
15784.906
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
511
</td>
<td style="text-align:right;">
14526.089
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:right;">
16685.077
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
480
</td>
<td style="text-align:right;">
15424.727
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:right;">
18735.705
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
439
</td>
<td style="text-align:right;">
17465.382
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:right;">
14355.531
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
571
</td>
<td style="text-align:right;">
15614.460
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:right;">
12414.754
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
637
</td>
<td style="text-align:right;">
13508.147
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:right;">
15532.036
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
528
</td>
<td style="text-align:right;">
17063.216
</td>
</tr>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:right;">
13183.154
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
609
</td>
<td style="text-align:right;">
14239.709
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
6621.366
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
954
</td>
<td style="text-align:right;">
6086.272
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:right;">
6175.261
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
975
</td>
<td style="text-align:right;">
5682.795
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:right;">
6319.006
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
977
</td>
<td style="text-align:right;">
5841.683
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:right;">
6628.684
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
929
</td>
<td style="text-align:right;">
6179.244
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:right;">
5792.873
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1018
</td>
<td style="text-align:right;">
6300.887
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:right;">
5870.551
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
1021
</td>
<td style="text-align:right;">
6387.583
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:right;">
6020.034
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
1002
</td>
<td style="text-align:right;">
6613.501
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="maxlfq_protein_intensity_data_wideformat-csv">maxLFQ_protein_intensity_data_wideFormat.csv<a class="anchor" aria-label="anchor" href="#maxlfq_protein_intensity_data_wideformat-csv"></a>
</h5>
<p>The <strong>maxLFQ_protein_intensity_data_wideFormat.csv</strong>
wide-tablular table contains the information about the calculated MaxLFQ
protein intensity data calculated by the <a href="https://academic.oup.com/bioinformatics/article/36/8/2611/5697917" class="external-link">iq
package</a> algorithm.</p>
<ul>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>other columns</u> column names = capped R.FileName; values =
MaxLFQ intensity</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR1
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR2
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR3
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR4
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR1
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR2
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR3
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR4
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
16246.423
</td>
<td style="text-align:right;">
15784.906
</td>
<td style="text-align:right;">
16685.077
</td>
<td style="text-align:right;">
18735.705
</td>
<td style="text-align:right;">
14355.531
</td>
<td style="text-align:right;">
12414.754
</td>
<td style="text-align:right;">
15532.036
</td>
<td style="text-align:right;">
13183.154
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
6621.366
</td>
<td style="text-align:right;">
6175.261
</td>
<td style="text-align:right;">
6319.006
</td>
<td style="text-align:right;">
6628.684
</td>
<td style="text-align:right;">
5792.873
</td>
<td style="text-align:right;">
5870.551
</td>
<td style="text-align:right;">
6020.034
</td>
<td style="text-align:right;">
5664.491
</td>
</tr>
<tr>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:right;">
270030.420
</td>
<td style="text-align:right;">
255377.914
</td>
<td style="text-align:right;">
250439.309
</td>
<td style="text-align:right;">
238487.251
</td>
<td style="text-align:right;">
107072.953
</td>
<td style="text-align:right;">
106952.527
</td>
<td style="text-align:right;">
107589.673
</td>
<td style="text-align:right;">
102970.133
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
31094.169
</td>
<td style="text-align:right;">
27795.500
</td>
<td style="text-align:right;">
29755.621
</td>
<td style="text-align:right;">
26191.666
</td>
<td style="text-align:right;">
23914.895
</td>
<td style="text-align:right;">
22749.815
</td>
<td style="text-align:right;">
22023.247
</td>
<td style="text-align:right;">
22871.072
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R8F8
</td>
<td style="text-align:right;">
11617.795
</td>
<td style="text-align:right;">
12296.678
</td>
<td style="text-align:right;">
10715.977
</td>
<td style="text-align:right;">
16699.834
</td>
<td style="text-align:right;">
8618.927
</td>
<td style="text-align:right;">
11483.775
</td>
<td style="text-align:right;">
10355.943
</td>
<td style="text-align:right;">
10387.003
</td>
</tr>
<tr>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
9823.646
</td>
<td style="text-align:right;">
10624.659
</td>
<td style="text-align:right;">
10481.575
</td>
<td style="text-align:right;">
10004.437
</td>
<td style="text-align:right;">
9040.597
</td>
<td style="text-align:right;">
8708.832
</td>
<td style="text-align:right;">
8357.038
</td>
<td style="text-align:right;">
8150.987
</td>
</tr>
<tr>
<td style="text-align:left;">
O00458
</td>
<td style="text-align:right;">
3035.682
</td>
<td style="text-align:right;">
4027.202
</td>
<td style="text-align:right;">
3914.859
</td>
<td style="text-align:right;">
4039.317
</td>
<td style="text-align:right;">
3032.950
</td>
<td style="text-align:right;">
2283.425
</td>
<td style="text-align:right;">
2717.885
</td>
<td style="text-align:right;">
2109.823
</td>
</tr>
<tr>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
25136.634
</td>
<td style="text-align:right;">
26433.396
</td>
<td style="text-align:right;">
25734.191
</td>
<td style="text-align:right;">
26915.220
</td>
<td style="text-align:right;">
21718.440
</td>
<td style="text-align:right;">
21335.804
</td>
<td style="text-align:right;">
21920.659
</td>
<td style="text-align:right;">
21584.259
</td>
</tr>
<tr>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
62322.848
</td>
<td style="text-align:right;">
60977.502
</td>
<td style="text-align:right;">
61372.144
</td>
<td style="text-align:right;">
61952.267
</td>
<td style="text-align:right;">
52451.533
</td>
<td style="text-align:right;">
54390.651
</td>
<td style="text-align:right;">
52178.827
</td>
<td style="text-align:right;">
53813.455
</td>
</tr>
<tr>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
6960.431
</td>
<td style="text-align:right;">
6992.641
</td>
<td style="text-align:right;">
6753.587
</td>
<td style="text-align:right;">
6652.533
</td>
<td style="text-align:right;">
5285.016
</td>
<td style="text-align:right;">
6100.743
</td>
<td style="text-align:right;">
5370.444
</td>
<td style="text-align:right;">
6248.809
</td>
</tr>
<tr>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
42004.038
</td>
<td style="text-align:right;">
42855.328
</td>
<td style="text-align:right;">
40434.184
</td>
<td style="text-align:right;">
43890.415
</td>
<td style="text-align:right;">
17571.980
</td>
<td style="text-align:right;">
17785.075
</td>
<td style="text-align:right;">
16900.070
</td>
<td style="text-align:right;">
18590.006
</td>
</tr>
<tr>
<td style="text-align:left;">
O13547
</td>
<td style="text-align:right;">
24870.131
</td>
<td style="text-align:right;">
23245.571
</td>
<td style="text-align:right;">
27541.161
</td>
<td style="text-align:right;">
21702.950
</td>
<td style="text-align:right;">
8801.724
</td>
<td style="text-align:right;">
8411.811
</td>
<td style="text-align:right;">
8651.519
</td>
<td style="text-align:right;">
9466.557
</td>
</tr>
<tr>
<td style="text-align:left;">
O14521
</td>
<td style="text-align:right;">
49214.326
</td>
<td style="text-align:right;">
49070.276
</td>
<td style="text-align:right;">
49341.867
</td>
<td style="text-align:right;">
51358.122
</td>
<td style="text-align:right;">
44134.583
</td>
<td style="text-align:right;">
33284.929
</td>
<td style="text-align:right;">
35259.856
</td>
<td style="text-align:right;">
32667.578
</td>
</tr>
<tr>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
19756.871
</td>
<td style="text-align:right;">
20191.191
</td>
<td style="text-align:right;">
18788.607
</td>
<td style="text-align:right;">
19367.470
</td>
<td style="text-align:right;">
16934.600
</td>
<td style="text-align:right;">
16137.321
</td>
<td style="text-align:right;">
15052.328
</td>
<td style="text-align:right;">
14846.953
</td>
</tr>
<tr>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
45989.129
</td>
<td style="text-align:right;">
46705.571
</td>
<td style="text-align:right;">
46212.956
</td>
<td style="text-align:right;">
45598.511
</td>
<td style="text-align:right;">
39090.096
</td>
<td style="text-align:right;">
38423.177
</td>
<td style="text-align:right;">
38617.010
</td>
<td style="text-align:right;">
38814.448
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="maxlfq_protein_intensity_rank_data_wideformat-csv">maxLFQ_protein_intensity_rank_data_wideFormat.csv<a class="anchor" aria-label="anchor" href="#maxlfq_protein_intensity_rank_data_wideformat-csv"></a>
</h5>
<p>The <strong>maxLFQ_protein_intensity_data_wideFormat.csv</strong>
wide-tablular table contains the information about the calculated MaxLFQ
protein intensity ranks.</p>
<ul>
<li>
<u>PG.ProteinGroups</u> protein group IDs</li>
<li>
<u>other columns</u> column names = capped R.FileName; values =
MaxLFQ intensity ranks</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR1
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR2
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR3
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixA_TR4
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR1
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR2
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR3
</th>
<th style="text-align:right;">
20230222_Exp…anual_MixB_TR4
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
497
</td>
<td style="text-align:right;">
511
</td>
<td style="text-align:right;">
480
</td>
<td style="text-align:right;">
439
</td>
<td style="text-align:right;">
571
</td>
<td style="text-align:right;">
637
</td>
<td style="text-align:right;">
528
</td>
<td style="text-align:right;">
609
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
954
</td>
<td style="text-align:right;">
975
</td>
<td style="text-align:right;">
977
</td>
<td style="text-align:right;">
929
</td>
<td style="text-align:right;">
1018
</td>
<td style="text-align:right;">
1021
</td>
<td style="text-align:right;">
1002
</td>
<td style="text-align:right;">
1025
</td>
</tr>
<tr>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
80
</td>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
79
</td>
<td style="text-align:right;">
86
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
294
</td>
<td style="text-align:right;">
314
</td>
<td style="text-align:right;">
296
</td>
<td style="text-align:right;">
334
</td>
<td style="text-align:right;">
382
</td>
<td style="text-align:right;">
400
</td>
<td style="text-align:right;">
402
</td>
<td style="text-align:right;">
401
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R8F8
</td>
<td style="text-align:right;">
661
</td>
<td style="text-align:right;">
628
</td>
<td style="text-align:right;">
703
</td>
<td style="text-align:right;">
479
</td>
<td style="text-align:right;">
818
</td>
<td style="text-align:right;">
668
</td>
<td style="text-align:right;">
717
</td>
<td style="text-align:right;">
716
</td>
</tr>
<tr>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
751
</td>
<td style="text-align:right;">
705
</td>
<td style="text-align:right;">
714
</td>
<td style="text-align:right;">
735
</td>
<td style="text-align:right;">
790
</td>
<td style="text-align:right;">
815
</td>
<td style="text-align:right;">
835
</td>
<td style="text-align:right;">
840
</td>
</tr>
<tr>
<td style="text-align:left;">
O00458
</td>
<td style="text-align:right;">
1287
</td>
<td style="text-align:right;">
1177
</td>
<td style="text-align:right;">
1176
</td>
<td style="text-align:right;">
1140
</td>
<td style="text-align:right;">
1276
</td>
<td style="text-align:right;">
1355
</td>
<td style="text-align:right;">
1317
</td>
<td style="text-align:right;">
1354
</td>
</tr>
<tr>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
339
</td>
<td style="text-align:right;">
332
</td>
<td style="text-align:right;">
338
</td>
<td style="text-align:right;">
321
</td>
<td style="text-align:right;">
412
</td>
<td style="text-align:right;">
417
</td>
<td style="text-align:right;">
406
</td>
<td style="text-align:right;">
418
</td>
</tr>
<tr>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
143
</td>
<td style="text-align:right;">
151
</td>
<td style="text-align:right;">
146
</td>
<td style="text-align:right;">
140
</td>
<td style="text-align:right;">
185
</td>
<td style="text-align:right;">
178
</td>
<td style="text-align:right;">
183
</td>
<td style="text-align:right;">
181
</td>
</tr>
<tr>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
925
</td>
<td style="text-align:right;">
917
</td>
<td style="text-align:right;">
940
</td>
<td style="text-align:right;">
926
</td>
<td style="text-align:right;">
1058
</td>
<td style="text-align:right;">
999
</td>
<td style="text-align:right;">
1051
</td>
<td style="text-align:right;">
979
</td>
</tr>
<tr>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
222
</td>
<td style="text-align:right;">
217
</td>
<td style="text-align:right;">
226
</td>
<td style="text-align:right;">
201
</td>
<td style="text-align:right;">
488
</td>
<td style="text-align:right;">
486
</td>
<td style="text-align:right;">
503
</td>
<td style="text-align:right;">
475
</td>
</tr>
<tr>
<td style="text-align:left;">
O13547
</td>
<td style="text-align:right;">
341
</td>
<td style="text-align:right;">
369
</td>
<td style="text-align:right;">
320
</td>
<td style="text-align:right;">
389
</td>
<td style="text-align:right;">
807
</td>
<td style="text-align:right;">
834
</td>
<td style="text-align:right;">
814
</td>
<td style="text-align:right;">
768
</td>
</tr>
<tr>
<td style="text-align:left;">
O14521
</td>
<td style="text-align:right;">
181
</td>
<td style="text-align:right;">
183
</td>
<td style="text-align:right;">
180
</td>
<td style="text-align:right;">
174
</td>
<td style="text-align:right;">
216
</td>
<td style="text-align:right;">
285
</td>
<td style="text-align:right;">
263
</td>
<td style="text-align:right;">
292
</td>
</tr>
<tr>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
417
</td>
<td style="text-align:right;">
412
</td>
<td style="text-align:right;">
444
</td>
<td style="text-align:right;">
423
</td>
<td style="text-align:right;">
499
</td>
<td style="text-align:right;">
520
</td>
<td style="text-align:right;">
542
</td>
<td style="text-align:right;">
553
</td>
</tr>
<tr>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
193
</td>
<td style="text-align:right;">
194
</td>
<td style="text-align:right;">
192
</td>
<td style="text-align:right;">
193
</td>
<td style="text-align:right;">
244
</td>
<td style="text-align:right;">
250
</td>
<td style="text-align:right;">
246
</td>
<td style="text-align:right;">
245
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="maxlfq_protein_intensity_data_normalization_factor-csv">maxLFQ_protein_intensity_data_normalization_factor.csv<a class="anchor" aria-label="anchor" href="#maxlfq_protein_intensity_data_normalization_factor-csv"></a>
</h5>
<p>The
<strong>maxLFQ_protein_intensity_data_normalization_factor.csv</strong>
table contains median-median normalization factors of MaxLFQ protein
intensity data</p>
<ul>
<li>
<u>R.FileName </u> protein group IDs</li>
<li>
<u>maxLFQ_post_calculation_normalization_factor</u> MaxLFQ
median-median normalization factor</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.FileName
</th>
<th style="text-align:right;">
maxLFQ_post_calculation_normalization_factor
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR1
</td>
<td style="text-align:right;">
1.0879181
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR2
</td>
<td style="text-align:right;">
1.0866591
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR3
</td>
<td style="text-align:right;">
1.0817097
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixA_TR4
</td>
<td style="text-align:right;">
1.0727338
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR1
</td>
<td style="text-align:right;">
0.9193742
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR2
</td>
<td style="text-align:right;">
0.9190568
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR3
</td>
<td style="text-align:right;">
0.9102643
</td>
</tr>
<tr>
<td style="text-align:left;">
20230222_Exp…anual_MixB_TR4
</td>
<td style="text-align:right;">
0.9258022
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="cv_20percent_cumulative_frequency-csv">CV_20percent_cumulative_frequency.csv<a class="anchor" aria-label="anchor" href="#cv_20percent_cumulative_frequency-csv"></a>
</h5>
<p>The <strong>CV_20percent_cumulative_frequency.csv</strong> table
contains the cumulative frequency of the CV≤0.2 fraction on peptide and
protein intensity level</p>
<ul>
<li>
<u>R.Condition </u> conditions as provided by Spectronaut
report</li>
<li>
<u>cumulative frequency</u> cumulative frequency</li>
<li>
<u>CV_cut</u> CV threshold for which the cumulative frequency was
calculated</li>
<li>
<u>level</u> peptide or protein level</li>
<li>
<u>cumulative frequency proportion</u> cumulative frequency
proportion on overall peptide or protein numbers per condition</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:right;">
cumulative frequency
</th>
<th style="text-align:right;">
CV_cut
</th>
<th style="text-align:left;">
level
</th>
<th style="text-align:right;">
cumulative frequency proportion
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:right;">
61892
</td>
<td style="text-align:right;">
0.2
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:right;">
0.5397072
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:right;">
64670
</td>
<td style="text-align:right;">
0.2
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:right;">
0.5639317
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:right;">
8192
</td>
<td style="text-align:right;">
0.2
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:right;">
0.8149622
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:right;">
8301
</td>
<td style="text-align:right;">
0.2
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:right;">
0.8258058
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="cv_cumulative_frequency-csv">CV_cumulative_frequency.csv<a class="anchor" aria-label="anchor" href="#cv_cumulative_frequency-csv"></a>
</h5>
<p>The comprehensive <strong>CV_cumulative_frequency.csv</strong> table
contains the cumulative frequency on peptide and protein intensity
level</p>
<ul>
<li>
<u>R.Condition </u> conditions as provided by Spectronaut
report</li>
<li>
<u>cumulative frequency</u> cumulative frequency</li>
<li>
<u>CV_cut</u> CV threshold for which the cumulative frequency was
calculated</li>
<li>
<u>level</u> peptide or protein level</li>
<li>
<u>cumulative frequency proportion</u> cumulative frequency
proportion on overall peptide or protein numbers per condition</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
R.Condition
</th>
<th style="text-align:left;">
cumulative frequency
</th>
<th style="text-align:left;">
CV_cut
</th>
<th style="text-align:left;">
level
</th>
<th style="text-align:left;">
cumulative frequency proportion
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
231
</td>
<td style="text-align:left;">
0.01
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.002014353
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
355
</td>
<td style="text-align:left;">
0.01
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.003095651
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
59
</td>
<td style="text-align:left;">
0.01
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.005869479
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
103
</td>
<td style="text-align:left;">
0.01
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.010246717071229606
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
1315
</td>
<td style="text-align:left;">
0.02
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.011466989893352634
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
1881
</td>
<td style="text-align:left;">
0.02
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.016402591626917342
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
341
</td>
<td style="text-align:left;">
0.02
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.033923597
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
527
</td>
<td style="text-align:left;">
0.02
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.052427378
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
3188
</td>
<td style="text-align:left;">
0.03
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.027799820365025246
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
4429
</td>
<td style="text-align:left;">
0.03
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.03862152
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
862
</td>
<td style="text-align:left;">
0.03
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.085754079
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
1175
</td>
<td style="text-align:left;">
0.03
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.11689216076402706
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
5835
</td>
<td style="text-align:left;">
0.04
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.050882042606625565
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
7877
</td>
<td style="text-align:left;">
0.04
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.068688577
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
1464
</td>
<td style="text-align:left;">
0.04
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.14564265817747712
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
1969
</td>
<td style="text-align:left;">
0.04
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.19588141663350578
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
8950
</td>
<td style="text-align:left;">
0.05
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.078045292
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
11753
</td>
<td style="text-align:left;">
0.05
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.10248785719891522
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
2126
</td>
<td style="text-align:left;">
0.05
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.21150019896538003
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
2681
</td>
<td style="text-align:left;">
0.05
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.2667130919220056
</td>
</tr>
<tr>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
…
</td>
<td style="text-align:left;">
…
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
114091
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.9948899953783235
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
114108
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
0.9950382378332185
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
10026
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.9974134500596896
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
10026
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
0.9974134500596896
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
114677
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
114677
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
peptide level
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix A
</td>
<td style="text-align:left;">
10052
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HYE mix B
</td>
<td style="text-align:left;">
10052
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
protein level
</td>
<td style="text-align:left;">
1
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Stephan Michalik.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

    </footer>
</div>





  </body>
</html>
