<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>SpectroPipeR - step 4 - statistics • SpectroPipeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.4.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="SpectroPipeR - step 4 - statistics">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">SpectroPipeR</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.3.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles">
<li><a class="dropdown-item" href="../articles/a01_SpectroPipeR_read_Spectronaut_data.html">SpectroPipeR - step 1 - read Spectronaut data</a></li>
    <li><a class="dropdown-item" href="../articles/a02_SpectroPipeR_norm_quant.html">SpectroPipeR - step 2 - normalization and quantification</a></li>
    <li><a class="dropdown-item" href="../articles/a03_SpectroPipeR_MVA_analysis.html">SpectroPipeR - step 3 -  MVA analysis</a></li>
    <li><a class="dropdown-item" href="../articles/a04_SpectroPipeR_statistics.html">SpectroPipeR - step 4 - statistics</a></li>
    <li><a class="dropdown-item" href="../articles/a05_SpectroPipeR_reporting.html">SpectroPipeR - step 5 - reporting</a></li>
    <li><a class="dropdown-item" href="../articles/a06_SpectroPipeR_XIC_plot.html">SpectroPipeR - additional feature - XIC plotting module</a></li>
    <li><a class="dropdown-item" href="../articles/a07_Gprofiler2_code_suggestion.html">SpectroPipeR - example code - functional enrichment analysis (external package)</a></li>
  </ul>
</li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">



<script src="a04_SpectroPipeR_statistics_files/kePrint-0.0.1/kePrint.js"></script><link href="a04_SpectroPipeR_statistics_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.png" class="logo" alt=""><h1>SpectroPipeR - step 4 - statistics</h1>
            
      

      <div class="d-none name"><code>a04_SpectroPipeR_statistics.Rmd</code></div>
    </div>

    
    
<div id="Tip1" class="greeting message" style="color: blue;">
<p><strong>If you want to have a closer look at the images and plots
just do a “right-click” on the image and select “open in new
tab”</strong></p>
</div>
<div id="Tip2" class="greeting message" style="color: green;">
<p><strong>The output folders contained figures that were always
exported in two file formats: vector PDF and raster PNG.</strong></p>
<p><u>Recommendations for MacOS Users</u></p>
<p>For MacOS users, it is recommended to use the PDF files, for example
in Keynote, to achieve better quality.</p>
<p><u>Recommendations for Windows Users</u></p>
<p>For Windows users, it is recommended to use the PNG files, as
applications like Microsoft PowerPoint can sometimes have issues when
working with vector graphics.</p>
</div>
<div class="section level2">
<h2 id="spectropiper-statistics">SpectroPipeR statistics<a class="anchor" aria-label="anchor" href="#spectropiper-statistics"></a>
</h2>
<p>The statistical analysis is carried out by utilizing the <a href="https://bioconductor.org/packages/release/bioc/html/PECA.html" class="external-link">PECA
package</a></p>
<p>The recommended test is the “rots” test (ROPECA approach). This
ROPECA (reproducibility-optimized peptide change averaging) approach can
be inspected in detail in <a href="https://www.nature.com/articles/s41598-017-05949-y" class="external-link">Enhanced
differential expression statistics for data-independent acquisition
proteomics</a></p>
<div class="section level3">
<h3 id="background-information">background information<a class="anchor" aria-label="anchor" href="#background-information"></a>
</h3>
<div class="section level4">
<h4 id="statistical-analysis">statistical analysis<a class="anchor" aria-label="anchor" href="#statistical-analysis"></a>
</h4>
<p>Pair-wise comparison can be carried out using an ordinary t-test
(“t”), modified t-test (“modt”), or reproducibility-optimized test
statistic (“rots”). The type of data aggregation can be either median
(“median”) or tukey (“tukey”) for calculating protein values. The test
may be performed paired or unpaired depending on your experimental
design.</p>
<p>PECA determines differential gene expression using directly the
peptide intensity measurements of proteomic datasets. An change between
two groups of samples is first calculated for each peptide in the
datasets. The protein-level changes are then defined as median/tukey
over the peptide level changes. For more details about the peptide-level
expression change averaging (PECA) procedure, see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1316121/" class="external-link">Elo <em>et
al.</em> (2005)</a>, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728531/" class="external-link">Laajala
<em>et al.</em> (2009)</a> and <a href="https://www.bioconductor.org/packages/release/bioc/html/PECA.html" class="external-link">Suomi
<em>et al.</em> (2024)</a></p>
<p>PECA calculates the peptide level changes using the ordinary or
modified t-statistic. The ordinary t-statistic is calculated using the
function rowttests in the Bioconductor genefilter package.</p>
<p><strong>If “modt” is selected modified t-statistic is calculated
using limma package (fast processing time):</strong></p>
<p>The modified t-statistic is calculated using the linear modeling
approach in the Bioconductor limma package.</p>
<p>The empirical Bayes moderated t-statistics test each individual
contrast equal to zero. For each protein, the moderated F-statistic
tests whether all the contrasts are zero. The F-statistic is an overall
test computed from the set of t-statistics for that peptide This is
exactly analogous the relationship between t-tests and F-statistics in
conventional anova, except that the residual mean squares have been
moderated between proteins.</p>
<p><strong>If “rots” is selected a Reproducibility-Optimized Test
Statistic (ROTS) is calculated using the PECA package (slower processing
time):</strong></p>
<p>The reproducibility-optimization procedure (ROTS) enables the
selection of a suitable gene ranking statistic directly from the given
dataset. The statistic is optimized among a family of t-type statistics
<math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>d</mi><mi>α</mi></msub><mo>=</mo><mrow><mo stretchy="true" form="prefix">|</mo><msub><mover><mi>x</mi><mo accent="true">¯</mo></mover><mn>1</mn></msub><mo>−</mo><msub><mover><mi>x</mi><mo accent="true">¯</mo></mover><mn>2</mn></msub><mo stretchy="true" form="postfix">|</mo></mrow><mi>/</mi><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>α</mi><mn>1</mn></msub><mo>+</mo><msub><mi>α</mi><mn>2</mn></msub><mo>*</mo><mi>s</mi><mo stretchy="true" form="postfix">)</mo></mrow></mrow><annotation encoding="application/x-tex">d_\alpha = |\overline{x}_1 - \overline{x}_2| / (\alpha_1 + \alpha_2*s)</annotation></semantics></math>,
where
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mo stretchy="true" form="prefix">|</mo><msub><mover><mi>x</mi><mo accent="true">¯</mo></mover><mn>1</mn></msub><mo>−</mo><msub><mover><mi>x</mi><mo accent="true">¯</mo></mover><mn>2</mn></msub><mo stretchy="true" form="postfix">|</mo></mrow><annotation encoding="application/x-tex">|\overline{x}_1 - \overline{x}_2|</annotation></semantics></math>
is the difference between the two group averages of normalized peptide
abundances,
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>α</mi><mn>1</mn></msub><annotation encoding="application/x-tex">\alpha_1</annotation></semantics></math>
and
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msub><mi>α</mi><mn>2</mn></msub><annotation encoding="application/x-tex">\alpha_2</annotation></semantics></math>
are non-negative parameters to be optimized, and
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>s</mi><annotation encoding="application/x-tex">s</annotation></semantics></math>
is the pooled standard error. The optimal statistic is determined by
maximizing the reproducibility Z-score
<math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>Z</mi><mi>k</mi></msub><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>d</mi><mi>α</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mo>=</mo><mrow><mo stretchy="true" form="prefix">(</mo><msub><mi>R</mi><mi>k</mi></msub><mo>*</mo><msub><mi>d</mi><mi>α</mi></msub><mo>−</mo><msubsup><mi>R</mi><mi>k</mi><mn>0</mn></msubsup><mo>*</mo><msub><mi>d</mi><mi>α</mi></msub><mo stretchy="true" form="postfix">)</mo></mrow><mi>/</mi><msub><mi>s</mi><mi>k</mi></msub><mo>*</mo><msub><mi>d</mi><mi>α</mi></msub></mrow><annotation encoding="application/x-tex">Z_k(d_\alpha) = (R_k  * d_\alpha - R^0_k * d_\alpha) / s_k * d_\alpha</annotation></semantics></math>
over a lattice of
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>α</mi><mn>1</mn></msub><mo>∈</mo><mo stretchy="false" form="prefix">{</mo><mn>0</mn><mo>,</mo><mn>0.01</mn><mo>,</mo><mi>.</mi><mi>.</mi><mi>.</mi><mo>,</mo><mn>5</mn><mo stretchy="false" form="postfix">}</mo></mrow><annotation encoding="application/x-tex">\alpha_1\in\{0,0.01, ...,5\}</annotation></semantics></math>
and
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>α</mi><mn>2</mn></msub><mo>∈</mo><mo stretchy="false" form="prefix">{</mo><mn>0</mn><mo>,</mo><mn>1</mn><mo stretchy="false" form="postfix">}</mo></mrow><annotation encoding="application/x-tex">\alpha_2\in\{0,1\}</annotation></semantics></math>,
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><mi>k</mi><mo>∈</mo><mo stretchy="false" form="prefix">{</mo><mn>0</mn><mo>,</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>.</mi><mi>.</mi><mi>.</mi><mo>,</mo><mi>F</mi><mo stretchy="false" form="postfix">}</mo></mrow><annotation encoding="application/x-tex">k \in \{0,1,2,...,F\}</annotation></semantics></math>,
where F is the total number of peptides in the data
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msubsup><mi>R</mi><mi>k</mi><mn>0</mn></msubsup><mo>*</mo><msub><mi>d</mi><mi>α</mi></msub></mrow><annotation encoding="application/x-tex">R^0_k * d_\alpha</annotation></semantics></math>
is the corresponding reproducibility in randomized datasets permuted
over samples and
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow><msub><mi>s</mi><mi>k</mi></msub><mo>*</mo><msub><mi>d</mi><mi>α</mi></msub></mrow><annotation encoding="application/x-tex">s_k * d_\alpha</annotation></semantics></math>
is the standard deviation of the bootstrap distribution. Reproducibility
is defined as the average overlap of
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mi>k</mi><annotation encoding="application/x-tex">k</annotation></semantics></math>
top-ranked peptides over pairs of bootstrapped datasets. For
protein-level inference of differential expression, the median of
peptide-level p-values is used as a score for each protein taking the
direction of change into account. The protein-level significance of the
detection is then calculated using beta distribution. Under the null
hypothesis, the p-values of the peptides follow the uniform distribution
U(0,1). Furthermore, the order statistics from U(0,1) distribution
follow a beta distribution. Finally, the FDR is calculated using the
Benjamini-Hochberg procedure.</p>
<p>This complex and computation intense procedure allow a more precise
estimate of significance than other methods. For further details please
see <a href="https://www.nature.com/articles/s41598-017-05949-y#Sec3https://www.nature.com/articles/s41598-017-05949-y#Sec3" class="external-link">Suomi
&amp; Elo 2017</a></p>
<p>The significance of an expression change is determined based on the
analytical p-value of the protein-level test statistic. Unadjusted
p-values are reported along with the corresponding p-values looked up
from beta ditribution. The quality control and filtering of the data
(e.g. based on low intensity or peptide specificity) is left to the
user.</p>
</div>
<div class="section level4">
<h4 id="effect-size-implementation-in-spectropiper">Effect size implementation in SpectroPipeR<a class="anchor" aria-label="anchor" href="#effect-size-implementation-in-spectropiper"></a>
</h4>
<p>In SpectroPipeR the effect size (Cohen’s d) of peptide ratios per
comparisons is implemented as follows. The peptide intensities were used
to calculate the mean scaled peptide intensities per protein. This step
is important since peptides have different intensities due to different
flyability in mass spec. For example if two peptides are injected at
10fmol and one flies much better than the other, then the better flying
peptide will have a much higher intensity. Therefore we need to adjust
for this effect and scale the peptides in the same numeric region.
Following the scaling process, the intensities of the scaled peptides
were employed to compute Cohen’s d for each protein, facilitating
subsequent comparative analysis.</p>
</div>
</div>
<div class="section level3">
<h3 id="example-code">example code<a class="anchor" aria-label="anchor" href="#example-code"></a>
</h3>
<p><strong>statistics_module() needs the output of the
norm_quant_module() !</strong></p>
<p>The condition_comparisons (mandatory) requires a cbind() with the
user specified conditions as used during the setup of the analysis in
Spectronaut. Therefore provide the conditions, which should be compared
in the R code.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># condition comparison example</span></span>
<span><span class="va">condition_comparisons_example</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span></span>
<span>                                <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"condition_1"</span>,<span class="st">"condition_control"</span><span class="op">)</span>,</span>
<span>                                <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"condition_2"</span>,<span class="st">"condition_control"</span><span class="op">)</span>,</span>
<span>                                <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"condition_3"</span>,<span class="st">"condition_control"</span><span class="op">)</span></span>
<span>                                <span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># step 4: statistics module</span></span>
<span><span class="va">SpectroPipeR_data_stats</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/statistics_module.html">statistics_module</a></span><span class="op">(</span>SpectroPipeR_data_quant <span class="op">=</span> <span class="va">SpectroPipeR_data_quant</span>,</span>
<span>                                       condition_comparisons <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"HYE mix A"</span>,<span class="st">"HYE mix B"</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">##*****************************************</span></span>
<span><span class="co">## STATISTICS MODULE</span></span>
<span><span class="co">##*****************************************</span></span>
<span><span class="co">#</span></span>
<span><span class="co">#reformatting data ...</span></span>
<span><span class="co">#register processor cores ...</span></span>
<span><span class="co">#performing statistical analysis (this might take a while) ...</span></span>
<span><span class="co">#  |=============================================================================================| #100%</span></span>
<span><span class="co">#close cores ...</span></span>
<span><span class="co">#</span></span>
<span><span class="co">#start to end time comparison for stat. analysis: 0.00111424499087863 hours</span></span>
<span><span class="co">#estimating effect sizes ...</span></span>
<span><span class="co"># [============================================================]  100.00% - calc. effect sizes... </span></span>
<span><span class="co">#</span></span>
<span><span class="co">#join and tidy tables ...</span></span>
<span><span class="co">#filtering statistical table using supplied cutoffs ...</span></span>
<span><span class="co">#writing output files ...</span></span>
<span><span class="co">#adding iBAQ quantiles to statistics table ...                                                                  # </span></span>
<span><span class="co">#generating Excel outputs ...                                                                                   # </span></span>
<span><span class="co">#performing fold-change cutoff sensitivity analysis ...                                                         # </span></span>
<span><span class="co">#plotting fold-change cutoff sensitivity analysis ...</span></span>
<span><span class="co"># [================================================================================]  100.00% - 1 </span></span>
<span><span class="co">#</span></span>
<span><span class="co">#plotting fold-change simple cutoff sensitivity analysis (peptide n &gt; 1)...</span></span>
<span><span class="co"># [================================================================================]  100.00% - 1 </span></span>
<span><span class="co">#</span></span>
<span><span class="co">#generating volcano plots ...</span></span>
<span><span class="co"># [============================================================]  100.00% - Volcano plots with adj. p-value </span></span>
<span><span class="co"># [============================================================]  100.00% - Volcano plots with adj. p-value </span></span>
<span><span class="co"># [============================================================]  100.00% - Volcano plots with raw p-value </span></span>
<span><span class="co">#</span></span>
<span><span class="co">#condition-comparison-wise signal to noise comparison ...</span></span>
<span><span class="co">#...signal to noise: save scatter plot...</span></span>
<span><span class="co">#condition-comparison-wise comparison of peptide-int.-ratios vs. protein-int.-ratios ...</span></span>
<span><span class="co">#...calculating protein ratios...</span></span>
<span><span class="co">#...combining ratio tables...</span></span>
<span><span class="co">#...generating ratio-ratios: protein_ratios/peptide_ratios...</span></span>
<span><span class="co">#...filter for at least 2 peptides...</span></span>
<span><span class="co">#...adding of protein intensity to table...</span></span>
<span><span class="co">#...add signal to noise per group...</span></span>
<span><span class="co">#...add detection with selected q-value cutoff with at least 2 peptides per replicate...</span></span>
<span><span class="co">#...add direction comparison for protein or peptide condition comp. ratio...</span></span>
<span><span class="co">#...save table of stat. significant with poor signal to noise...</span></span>
<span><span class="co">#...counting protein which having a 2fold difference...</span></span>
<span><span class="co">#...select Top15 over- or under-estimated proteins...</span></span>
<span><span class="co">#...protein int. benchmark: save scatter plot...</span></span>
<span><span class="co">#...protein int. benchmark: save histogram plot...</span></span>
<span><span class="co">#...protein int. benchmark: save table...</span></span>
<span><span class="co">#...counting protein: gradient of difference...                                                                 # </span></span>
<span><span class="co">#...protein int. benchmark: gradient of difference area plots...</span></span>
<span><span class="co">#statistical analysis module done --&gt; please check outputs in folder: ../SpectroPipeR_test_folder/06_statistics/  </span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="statistics_module-outputs">statistics_module() outputs<a class="anchor" aria-label="anchor" href="#statistics_module-outputs"></a>
</h3>
<p>The output in your specified output folder for the
norm_quant_module() function should look like in this example
(06_statistics, 05_processed_data):</p>
<p><img src="figures/folder_structure_statistics.png" style="width:30.0%"><img src="figures/folder_structure_processed.png" style="width:30.0%"></p>
<!-- figures statistics Spectronaut data -->
<div class="section level4">
<h4 class="tabset tabset-pills" id="statistics---figures">statistics - figures<a class="anchor" aria-label="anchor" href="#statistics---figures"></a>
</h4>
<div class="section level5">
<h5 id="volcano-plots">volcano plots<a class="anchor" aria-label="anchor" href="#volcano-plots"></a>
</h5>
<p>The <strong>volcano_plots_raw_p_value…</strong> illustrates the
volcano plot (raw p-value) of the statistical analysis for a specific
comparison.</p>
<p>The blue color indicates lower abundance and the orange indicates
higher abundance of the protein in regard to their peptide ratios.</p>
<p>The blue, grey or orange label depicts the number of proteins in each
fraction.</p>
<p>The user specified p-value and fold-change threshold are used to
determine the fractions.</p>
<p>On the right panel of the plot the Top10 (based on euclidean
distance) abundance differences are highlighted for the lower and upper
fraction.</p>
<p><img src="figures/04__volcano_plots_raw_p_value__B_manual_vs_A_manual.png" style="width:80.0%"></p>
<p>The <strong>volcano_plots_adjusted_p_value…</strong> illustrates the
volcano plot (adjusted p-value / q-value) of the statistical analysis
for a specific comparison.</p>
<p><img src="figures/04__volcano_plots_adjusted_p_value__B_manual_vs_A_manual.png" style="width:80.0%"></p>
<p>The
<strong>volcano_plots_effect_size_shape_adjusted_p_value…</strong>
illustrates the volcano plot (adjusted p-value / q-value) of the
statistical analysis for a specific comparison. The point shape depicts
the estimated effect size.</p>
<p><img src="figures/04__volcano_plots_effect_size_shape_q.png" style="width:80.0%"></p>
</div>
<div class="section level5">
<h5 id="cut-off-plots">cut-off plots<a class="anchor" aria-label="anchor" href="#cut-off-plots"></a>
</h5>
<p>The <strong>cutoff_test…</strong> &amp;
<strong>cutoff_simple_test…</strong> illustrate the protein count by
varying the fold-change threshold to filter significant proteins. They
should help to estimate a meaningful project specific fold-change cutoff
for the statistical analysis.</p>
<div class="float">
<img src="figures/04__cutoff_simple_test__B_manual_vs_A_manual.png" style="width:60.0%" alt="The simple cutoff plot highlights the protein count differences using various fold-change threshold with and without q-value filtering"><div class="figcaption">The simple cutoff plot highlights the protein
count differences using various fold-change threshold with and without
q-value filtering</div>
</div>
<div class="float">
<img src="figures/04__cutoff_test__B_manual_vs_A_manual.png" style="width:90.0%" alt="The cutoff plot highlights the protein count (with 1 or at least 2 peptides) differences using various fold-change threshold with and without q-value filtering"><div class="figcaption">The cutoff plot highlights the protein count
(with 1 or at least 2 peptides) differences using various fold-change
threshold with and without q-value filtering</div>
</div>
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="statistics---tables">statistics - tables<a class="anchor" aria-label="anchor" href="#statistics---tables"></a>
</h4>
<div class="section level5">
<h5 id="statistical_analysis-csv">statistical_analysis.csv<a class="anchor" aria-label="anchor" href="#statistical_analysis-csv"></a>
</h5>
<p>The <strong>statistical_analysis.csv</strong> holds the information
of the statistical analysis.</p>
<ul>
<li>
<u>slr:</u> signal log2-ratios on peptide basis</li>
<li>
<u>t:</u> t of t-statistics on peptide basis</li>
<li>
<u>score:</u> score of t-statistics on peptide basis</li>
<li>
<u>n:</u> number of peptides</li>
<li>
<u>p:</u> raw p-value of statistics on peptide basis</li>
<li>
<u>p.fdr:</u> adjusted p-value (q-value) of statistics on peptide
basis</li>
<li>
<u>PG.ProteinGroups:</u> Protein groups</li>
<li>
<u>group1:</u> group1 of condition comparison</li>
<li>
<u>group2:</u> group2 of condition comparison</li>
<li>
<u>slr_ratio_meta:</u> condition comparison; how the ratio is
formed</li>
<li>
<u>test:</u> which test was used for statistics on peptide
level</li>
<li>
<u>type:</u> which type of ratio aggregation to ProteinGroup level
was used for signal log2-ratios on peptide basis</li>
<li>
<u>significant_changed:</u> if there is a significant change FC
&amp; q-value (cutoffs e.g.: FC = 1.5 &amp; adjusted-p-value =
0.05)</li>
<li>
<u>significant_changed_raw_p:</u> if there is a significant change
FC &amp; p-value (cutoffs e.g.: FC = 1.5 &amp; p-value = 0.05)</li>
<li>
<u>significant_changed_fc:</u> fold-change cutoff used for
analysis</li>
<li>
<u>significant_changed_p_value:</u> p-value/q-value cutoff used for
analysis</li>
<li>
<u>fold_change_absolute:</u> ablsolute fold-change</li>
<li>
<u>fold_change_direction:</u> fold-change direction</li>
<li>
<u>fold_change:</u> fold-change</li>
<li>
<u>effect_size_method:</u> effect size estimation method used</li>
<li>
<u>d:</u> effect size estimate</li>
<li>
<u>d_pooled_SD:</u> effect size estimate; pooled SD</li>
<li>
<u>d_95CI_lower:</u> effect size estimate: the lower 95% confidence
interval</li>
<li>
<u>d_95CI_upper:</u> effect size estimate: the upper 95% confidence
interval</li>
<li>
<u>d_magnitute:</u> a qualitative assessment of the magnitude of
effect size (|d|&lt;0.2 negligible, |d|&lt;0.5 small, |d|&lt;0.8 medium,
otherwise large); Cohen 1992</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:right;">
slr
</th>
<th style="text-align:right;">
t
</th>
<th style="text-align:right;">
score
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
p
</th>
<th style="text-align:right;">
p.fdr
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:left;">
slr_ratio_meta
</th>
<th style="text-align:left;">
test
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:left;">
significant_changed
</th>
<th style="text-align:left;">
significant_changed_raw_p
</th>
<th style="text-align:right;">
significant_changed_fc
</th>
<th style="text-align:right;">
significant_changed_p_value
</th>
<th style="text-align:right;">
fold_change_absolute
</th>
<th style="text-align:left;">
fold_change_direction
</th>
<th style="text-align:right;">
fold_change
</th>
<th style="text-align:left;">
effect_size_method
</th>
<th style="text-align:right;">
d
</th>
<th style="text-align:right;">
d_pooled_SD
</th>
<th style="text-align:right;">
d_95CI_lower
</th>
<th style="text-align:right;">
d_95CI_upper
</th>
<th style="text-align:left;">
d_magnitute
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:right;">
-0.0473512
</td>
<td style="text-align:right;">
-0.4068343
</td>
<td style="text-align:right;">
0.6962885
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.7433455
</td>
<td style="text-align:right;">
0.9010067
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.033366
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.033366
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2791355
</td>
<td style="text-align:right;">
0.1074963
</td>
<td style="text-align:right;">
-1.3567385
</td>
<td style="text-align:right;">
0.7984675
</td>
<td style="text-align:left;">
small
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0443521
</td>
<td style="text-align:right;">
0.4408390
</td>
<td style="text-align:right;">
0.6726308
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.8641528
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0164415
</td>
<td style="text-align:right;">
0.1860178
</td>
<td style="text-align:right;">
-0.4536598
</td>
<td style="text-align:right;">
0.4865428
</td>
<td style="text-align:left;">
negligible
</td>
</tr>
<tr>
<td style="text-align:right;">
-1.0186955
</td>
<td style="text-align:right;">
-21.5749594
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
0.0000003
</td>
<td style="text-align:left;">
A5Z2X5
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
2.026086
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-2.026086
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-15.0652499
</td>
<td style="text-align:right;">
0.0450459
</td>
<td style="text-align:right;">
-24.4420988
</td>
<td style="text-align:right;">
-5.6884009
</td>
<td style="text-align:left;">
large
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1165372
</td>
<td style="text-align:right;">
-1.0692391
</td>
<td style="text-align:right;">
0.3204605
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.1911809
</td>
<td style="text-align:right;">
0.2713360
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.084130
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.084130
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6380376
</td>
<td style="text-align:right;">
0.1449498
</td>
<td style="text-align:right;">
-1.2942932
</td>
<td style="text-align:right;">
0.0182179
</td>
<td style="text-align:left;">
medium
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0794428
</td>
<td style="text-align:right;">
-0.5016872
</td>
<td style="text-align:right;">
0.6312874
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.6312874
</td>
<td style="text-align:right;">
0.7874067
</td>
<td style="text-align:left;">
L0R8F8
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.056610
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.056610
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.3732878
</td>
<td style="text-align:right;">
0.1749949
</td>
<td style="text-align:right;">
-2.1185192
</td>
<td style="text-align:right;">
1.3719437
</td>
<td style="text-align:left;">
small
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0066561
</td>
<td style="text-align:right;">
0.1008351
</td>
<td style="text-align:right;">
0.9225103
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0.9999290
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1792522
</td>
<td style="text-align:right;">
0.1897052
</td>
<td style="text-align:right;">
-0.2454271
</td>
<td style="text-align:right;">
0.6039315
</td>
<td style="text-align:left;">
negligible
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.3342264
</td>
<td style="text-align:right;">
-2.2225784
</td>
<td style="text-align:right;">
0.0616644
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.0616644
</td>
<td style="text-align:right;">
0.0942844
</td>
<td style="text-align:left;">
O00458
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.260701
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.260701
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-1.5357510
</td>
<td style="text-align:right;">
0.1479843
</td>
<td style="text-align:right;">
-3.5045774
</td>
<td style="text-align:right;">
0.4330754
</td>
<td style="text-align:left;">
large
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0079990
</td>
<td style="text-align:right;">
-0.0748142
</td>
<td style="text-align:right;">
0.9424568
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9999709
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0573662
</td>
<td style="text-align:right;">
0.3145541
</td>
<td style="text-align:right;">
-0.3878923
</td>
<td style="text-align:right;">
0.5026247
</td>
<td style="text-align:left;">
negligible
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0022888
</td>
<td style="text-align:right;">
-0.0321861
</td>
<td style="text-align:right;">
0.9752227
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.001588
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.001588
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.0370422
</td>
<td style="text-align:right;">
0.1975283
</td>
<td style="text-align:right;">
-0.2794375
</td>
<td style="text-align:right;">
0.2053530
</td>
<td style="text-align:left;">
negligible
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1181271
</td>
<td style="text-align:right;">
-0.6426256
</td>
<td style="text-align:right;">
0.5409473
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.5998730
</td>
<td style="text-align:right;">
0.7532240
</td>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.085325
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.085325
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1723566
</td>
<td style="text-align:right;">
0.3216608
</td>
<td style="text-align:right;">
-0.2986087
</td>
<td style="text-align:right;">
0.6433219
</td>
<td style="text-align:left;">
negligible
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.9757784
</td>
<td style="text-align:right;">
-8.2918171
</td>
<td style="text-align:right;">
0.0000727
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.966702
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.966702
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-2.1848927
</td>
<td style="text-align:right;">
0.3573362
</td>
<td style="text-align:right;">
-2.7789090
</td>
<td style="text-align:right;">
-1.5908763
</td>
<td style="text-align:left;">
large
</td>
</tr>
<tr>
<td style="text-align:right;">
-1.2215582
</td>
<td style="text-align:right;">
-15.3916606
</td>
<td style="text-align:right;">
0.0000012
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.0000012
</td>
<td style="text-align:right;">
0.0000025
</td>
<td style="text-align:left;">
O13547
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
2.331985
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-2.331985
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-7.9147819
</td>
<td style="text-align:right;">
0.1013292
</td>
<td style="text-align:right;">
-13.0563463
</td>
<td style="text-align:right;">
-2.7732176
</td>
<td style="text-align:left;">
large
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.2271634
</td>
<td style="text-align:right;">
-2.2672522
</td>
<td style="text-align:right;">
0.0577325
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.0577325
</td>
<td style="text-align:right;">
0.0887239
</td>
<td style="text-align:left;">
O14521
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.170531
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.170531
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-1.5222502
</td>
<td style="text-align:right;">
0.0984701
</td>
<td style="text-align:right;">
-3.4871492
</td>
<td style="text-align:right;">
0.4426487
</td>
<td style="text-align:left;">
large
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0529473
</td>
<td style="text-align:right;">
-0.9991841
</td>
<td style="text-align:right;">
0.3510038
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.2831211
</td>
<td style="text-align:right;">
0.3925563
</td>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.037382
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.037382
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6291879
</td>
<td style="text-align:right;">
0.2031349
</td>
<td style="text-align:right;">
-1.4965383
</td>
<td style="text-align:right;">
0.2381626
</td>
<td style="text-align:left;">
medium
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0104878
</td>
<td style="text-align:right;">
-0.2441598
</td>
<td style="text-align:right;">
0.8141130
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9892582
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.007296
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.007296
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.4053804
</td>
<td style="text-align:right;">
0.1706905
</td>
<td style="text-align:right;">
-0.0443356
</td>
<td style="text-align:right;">
0.8550963
</td>
<td style="text-align:left;">
small
</td>
</tr>
</tbody>
</table>
<p>The files <strong>statistical_analysis_filtered…</strong> contain the
same information but are filtered for:</p>
<table class="table">
<colgroup>
<col width="26%">
<col width="73%">
</colgroup>
<thead><tr class="header">
<th>file</th>
<th>filter criteria</th>
</tr></thead>
<tbody>
<tr class="odd">
<td>statistical_analysis_filtered<br>__fc_1.5__raw_p_value_0.05.csv</td>
<td>user-specified abs. fold-change (e.g. 1.5) and raw p-value
(e.g. 0.05)</td>
</tr>
<tr class="even">
<td>statistical_analysis_filtered<br>__fc_1.5__raw_p_value_0.05<br>_2_more_peptides_per_protein.csv</td>
<td>user-specified abs. fold-change (e.g. 1.5) and raw p-value
(e.g. 0.05) and at least 2 peptides</td>
</tr>
<tr class="odd">
<td>statistical_analysis_filtered<br>__fc_1.5__adjusted_p_value_0.05.csv</td>
<td>user-specified abs. fold-change (e.g. 1.5) and adjusted p-value
(e.g. 0.05)</td>
</tr>
<tr class="even">
<td>statistical_analysis_filtered<br>__fc_1.5__adjusted_p_value_0.05<br>_2_more_peptides_per_protein.csv</td>
<td>user-specified abs. fold-change (e.g. 1.5) and adjusted p-value
(e.g. 0.05) and at least 2 peptides</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="statistical_analysis_2_more_peptides_per_protein_ibaq_quantiles-xlsx">statistical_analysis_2_more_peptides_per_protein_iBAQ_quantiles.xlsx<a class="anchor" aria-label="anchor" href="#statistical_analysis_2_more_peptides_per_protein_ibaq_quantiles-xlsx"></a>
</h5>
<p>The Excel table
<strong>statistical_analysis_2_more_peptides_per_protein_iBAQ_quantiles.xlsx</strong>
(csv file also available) holds the information of the statistical
analysis and the iBAQ quantiles associated with the comparisons.</p>
<div class="float">
<img src="figures/04__iBAQ_quantile_ratio_comparison_legend.png" style="width:40.0%" alt="iBAQ quantile ratio legend"><div class="figcaption">iBAQ quantile ratio legend</div>
</div>
<ul>
<li>
<u>slr:</u> signal log2-ratios on peptide basis</li>
<li>
<u>iBAQ_quantile_comp:</u> iBAQ quantiles of comparison</li>
<li>
<u>t:</u> t of t-statistics on peptide basis</li>
<li>
<u>score:</u> score of t-statistics on peptide basis</li>
<li>
<u>n:</u> number of peptides</li>
<li>
<u>p:</u> raw p-value of statistics on peptide basis</li>
<li>
<u>p.fdr:</u> adjusted p-value (q-value) of statistics on peptide
basis</li>
<li>
<u>PG.ProteinGroups:</u> Protein groups</li>
<li>
<u>group1:</u> group1 of condition comparison</li>
<li>
<u>group2:</u> group2 of condition comparison</li>
<li>
<u>slr_ratio_meta:</u> condition comparison; how the ratio is
formed</li>
<li>
<u>test:</u> which test was used for statistics on peptide
level</li>
<li>
<u>type:</u> which type of ratio aggregation to ProteinGroup level
was used for signal log2-ratios on peptide basis</li>
<li>
<u>significant_changed:</u> if there is a significant change FC
&amp; q-value (cutoffs e.g.: FC = 1.5 &amp; adjusted-p-value =
0.05)</li>
<li>
<u>significant_changed_raw_p:</u> if there is a significant change
FC &amp; p-value (cutoffs e.g.: FC = 1.5 &amp; p-value = 0.05)</li>
<li>
<u>significant_changed_fc:</u> fold-change cutoff used for
analysis</li>
<li>
<u>significant_changed_p_value:</u> p-value/q-value cutoff used for
analysis</li>
<li>
<u>fold_change_absolute:</u> ablsolute fold-change</li>
<li>
<u>fold_change_direction:</u> fold-change direction</li>
<li>
<u>fold_change:</u> fold-change</li>
<li>
<u>effect_size_method:</u> effect size estimation method used</li>
<li>
<u>d:</u> effect size estimate</li>
<li>
<u>d_pooled_SD:</u> effect size estimate; pooled SD</li>
<li>
<u>d_95CI_lower:</u> effect size estimate: the lower 95% confidence
interval</li>
<li>
<u>d_95CI_upper:</u> effect size estimate: the upper 95% confidence
interval</li>
<li>
<u>d_magnitute:</u> a qualitative assessment of the magnitude of
effect size (|d|&lt;0.2 negligible, |d|&lt;0.5 small, |d|&lt;0.8 medium,
otherwise large); Cohen 1992</li>
<li>
<u>group1__mean_iBAQ:</u> mean iBAQ intensity of group 1</li>
<li>
<u>group_1__iBAQ_quantiles:</u> iBAQ intensity quantile of group
1</li>
<li>
<u>group2__mean_iBAQ:</u> mean iBAQ intensity of group 2</li>
<li>
<u>group_2__iBAQ_quantiles:</u> iBAQ intensity quantile of group
2</li>
</ul>
<p>The
<strong>statistical_analysis_2_more_peptides_per_protein_iBAQ_quantiles.xlsx</strong>
holds information of the statistical analysis.</p>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:right;">
slr
</th>
<th style="text-align:left;">
iBAQ_quantile_comp
</th>
<th style="text-align:right;">
t
</th>
<th style="text-align:right;">
score
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
p
</th>
<th style="text-align:right;">
p.fdr
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:left;">
slr_ratio_meta
</th>
<th style="text-align:left;">
test
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:left;">
significant_changed
</th>
<th style="text-align:left;">
significant_changed_raw_p
</th>
<th style="text-align:right;">
significant_changed_fc
</th>
<th style="text-align:right;">
significant_changed_p_value
</th>
<th style="text-align:right;">
fold_change_absolute
</th>
<th style="text-align:left;">
fold_change_direction
</th>
<th style="text-align:right;">
fold_change
</th>
<th style="text-align:left;">
effect_size_method
</th>
<th style="text-align:right;">
d
</th>
<th style="text-align:right;">
d_pooled_SD
</th>
<th style="text-align:right;">
d_95CI_lower
</th>
<th style="text-align:right;">
d_95CI_upper
</th>
<th style="text-align:left;">
d_magnitute
</th>
<th style="text-align:right;">
group1__mean_iBAQ
</th>
<th style="text-align:left;">
group_1__iBAQ_quantiles
</th>
<th style="text-align:right;">
group2__mean_iBAQ
</th>
<th style="text-align:left;">
group_2__iBAQ_quantiles
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:right;">
-0.0473512
</td>
<td style="text-align:left;">
Q4/Q5
</td>
<td style="text-align:right;">
-0.4068343
</td>
<td style="text-align:right;">
0.6962885
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.7433455
</td>
<td style="text-align:right;">
0.9010067
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.033366
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.033366
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2791355
</td>
<td style="text-align:right;">
0.1074963
</td>
<td style="text-align:right;">
-1.3567385
</td>
<td style="text-align:right;">
0.7984675
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
2588.925
</td>
<td style="text-align:left;">
Q4
</td>
<td style="text-align:right;">
2660.450
</td>
<td style="text-align:left;">
Q5
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0443521
</td>
<td style="text-align:left;">
Q3/Q4
</td>
<td style="text-align:right;">
0.4408390
</td>
<td style="text-align:right;">
0.6726308
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.8641528
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0164415
</td>
<td style="text-align:right;">
0.1860178
</td>
<td style="text-align:right;">
-0.4536598
</td>
<td style="text-align:right;">
0.4865428
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
1404.825
</td>
<td style="text-align:left;">
Q3
</td>
<td style="text-align:right;">
1324.475
</td>
<td style="text-align:left;">
Q4
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1165372
</td>
<td style="text-align:left;">
Q8/Q9
</td>
<td style="text-align:right;">
-1.0692391
</td>
<td style="text-align:right;">
0.3204605
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.1911809
</td>
<td style="text-align:right;">
0.2713360
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.084130
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.084130
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6380376
</td>
<td style="text-align:right;">
0.1449498
</td>
<td style="text-align:right;">
-1.2942932
</td>
<td style="text-align:right;">
0.0182179
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
28498.150
</td>
<td style="text-align:left;">
Q8
</td>
<td style="text-align:right;">
31907.450
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0066561
</td>
<td style="text-align:left;">
Q5/Q6
</td>
<td style="text-align:right;">
0.1008351
</td>
<td style="text-align:right;">
0.9225103
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0.9999290
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1792522
</td>
<td style="text-align:right;">
0.1897052
</td>
<td style="text-align:right;">
-0.2454271
</td>
<td style="text-align:right;">
0.6039315
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
4894.400
</td>
<td style="text-align:left;">
Q5
</td>
<td style="text-align:right;">
4884.950
</td>
<td style="text-align:left;">
Q6
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0079990
</td>
<td style="text-align:left;">
Q8/Q9
</td>
<td style="text-align:right;">
-0.0748142
</td>
<td style="text-align:right;">
0.9424568
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9999709
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0573662
</td>
<td style="text-align:right;">
0.3145541
</td>
<td style="text-align:right;">
-0.3878923
</td>
<td style="text-align:right;">
0.5026247
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
30011.075
</td>
<td style="text-align:left;">
Q8
</td>
<td style="text-align:right;">
32368.550
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0022888
</td>
<td style="text-align:left;">
Q10/Q10
</td>
<td style="text-align:right;">
-0.0321861
</td>
<td style="text-align:right;">
0.9752227
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.001588
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.001588
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.0370422
</td>
<td style="text-align:right;">
0.1975283
</td>
<td style="text-align:right;">
-0.2794375
</td>
<td style="text-align:right;">
0.2053530
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
104792.700
</td>
<td style="text-align:left;">
Q10
</td>
<td style="text-align:right;">
112171.575
</td>
<td style="text-align:left;">
Q10
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1181271
</td>
<td style="text-align:left;">
Q5/Q5
</td>
<td style="text-align:right;">
-0.6426256
</td>
<td style="text-align:right;">
0.5409473
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.5998730
</td>
<td style="text-align:right;">
0.7532240
</td>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.085325
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.085325
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1723566
</td>
<td style="text-align:right;">
0.3216608
</td>
<td style="text-align:right;">
-0.2986087
</td>
<td style="text-align:right;">
0.6433219
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
3684.925
</td>
<td style="text-align:left;">
Q5
</td>
<td style="text-align:right;">
3548.625
</td>
<td style="text-align:left;">
Q5
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.9757784
</td>
<td style="text-align:left;">
Q10/Q10
</td>
<td style="text-align:right;">
-8.2918171
</td>
<td style="text-align:right;">
0.0000727
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.966702
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.966702
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-2.1848927
</td>
<td style="text-align:right;">
0.3573362
</td>
<td style="text-align:right;">
-2.7789090
</td>
<td style="text-align:right;">
-1.5908763
</td>
<td style="text-align:left;">
large
</td>
<td style="text-align:right;">
138334.725
</td>
<td style="text-align:left;">
Q10
</td>
<td style="text-align:right;">
251255.500
</td>
<td style="text-align:left;">
Q10
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0529473
</td>
<td style="text-align:left;">
Q8/Q9
</td>
<td style="text-align:right;">
-0.9991841
</td>
<td style="text-align:right;">
0.3510038
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.2831211
</td>
<td style="text-align:right;">
0.3925563
</td>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.037382
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.037382
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6291879
</td>
<td style="text-align:right;">
0.2031349
</td>
<td style="text-align:right;">
-1.4965383
</td>
<td style="text-align:right;">
0.2381626
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
26367.525
</td>
<td style="text-align:left;">
Q8
</td>
<td style="text-align:right;">
26587.425
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0104878
</td>
<td style="text-align:left;">
Q9/Q9
</td>
<td style="text-align:right;">
-0.2441598
</td>
<td style="text-align:right;">
0.8141130
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9892582
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.007296
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.007296
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.4053804
</td>
<td style="text-align:right;">
0.1706905
</td>
<td style="text-align:right;">
-0.0443356
</td>
<td style="text-align:right;">
0.8550963
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
44033.250
</td>
<td style="text-align:left;">
Q9
</td>
<td style="text-align:right;">
43786.775
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0820429
</td>
<td style="text-align:left;">
Q5/Q6
</td>
<td style="text-align:right;">
-1.0571006
</td>
<td style="text-align:right;">
0.3255960
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.1787867
</td>
<td style="text-align:right;">
0.2559203
</td>
<td style="text-align:left;">
O14657
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.058516
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.058516
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1920303
</td>
<td style="text-align:right;">
0.2575590
</td>
<td style="text-align:right;">
-0.3903803
</td>
<td style="text-align:right;">
0.7744409
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
4962.650
</td>
<td style="text-align:left;">
Q5
</td>
<td style="text-align:right;">
4795.525
</td>
<td style="text-align:left;">
Q6
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0063245
</td>
<td style="text-align:left;">
Q10/Q10
</td>
<td style="text-align:right;">
-0.0259550
</td>
<td style="text-align:right;">
0.9800181
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14818
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.004393
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.004393
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.0140356
</td>
<td style="text-align:right;">
0.2473297
</td>
<td style="text-align:right;">
-0.3885587
</td>
<td style="text-align:right;">
0.3604874
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
163847.650
</td>
<td style="text-align:left;">
Q10
</td>
<td style="text-align:right;">
162511.925
</td>
<td style="text-align:left;">
Q10
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0442411
</td>
<td style="text-align:left;">
Q3/Q4
</td>
<td style="text-align:right;">
-0.8248106
</td>
<td style="text-align:right;">
0.4366954
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.4196139
</td>
<td style="text-align:right;">
0.5551758
</td>
<td style="text-align:left;">
O14893
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.031141
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.031141
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2370409
</td>
<td style="text-align:right;">
0.1437852
</td>
<td style="text-align:right;">
-1.3131936
</td>
<td style="text-align:right;">
0.8391119
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
1305.125
</td>
<td style="text-align:left;">
Q3
</td>
<td style="text-align:right;">
1411.100
</td>
<td style="text-align:left;">
Q4
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0491310
</td>
<td style="text-align:left;">
Q8/Q9
</td>
<td style="text-align:right;">
0.5638555
</td>
<td style="text-align:right;">
0.5904612
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0.7406270
</td>
<td style="text-align:right;">
0.8992711
</td>
<td style="text-align:left;">
O14929
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.034641
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.034641
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.6283915
</td>
<td style="text-align:right;">
0.1706648
</td>
<td style="text-align:right;">
0.2132161
</td>
<td style="text-align:right;">
1.0435668
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
32077.275
</td>
<td style="text-align:left;">
Q8
</td>
<td style="text-align:right;">
30418.775
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0534097
</td>
<td style="text-align:left;">
Q8/Q9
</td>
<td style="text-align:right;">
1.1650318
</td>
<td style="text-align:right;">
0.2821887
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.2317136
</td>
<td style="text-align:right;">
0.3251779
</td>
<td style="text-align:left;">
O14949
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.037715
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.037715
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.9962874
</td>
<td style="text-align:right;">
0.0374448
</td>
<td style="text-align:right;">
-0.1406891
</td>
<td style="text-align:right;">
2.1332639
</td>
<td style="text-align:left;">
large
</td>
<td style="text-align:right;">
26784.625
</td>
<td style="text-align:left;">
Q8
</td>
<td style="text-align:right;">
25997.875
</td>
<td style="text-align:left;">
Q9
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0265630
</td>
<td style="text-align:left;">
Q3/Q3
</td>
<td style="text-align:right;">
-0.2641864
</td>
<td style="text-align:right;">
0.7992497
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0.9907917
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14981
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.018583
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.018583
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2876621
</td>
<td style="text-align:right;">
0.2581572
</td>
<td style="text-align:right;">
-0.6950461
</td>
<td style="text-align:right;">
0.1197219
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
1097.700
</td>
<td style="text-align:left;">
Q3
</td>
<td style="text-align:right;">
1053.100
</td>
<td style="text-align:left;">
Q3
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level5">
<h5 id="statistical_analysis_wide_format_2_more_peptides_per_protein-csvxlsx">statistical_analysis_WIDE_FORMAT_2_more_peptides_per_protein.csv/xlsx<a class="anchor" aria-label="anchor" href="#statistical_analysis_wide_format_2_more_peptides_per_protein-csvxlsx"></a>
</h5>
<p>The
<strong>statistical_analysis_WIDE_FORMAT_2_more_peptides_per_protein</strong>
contains the information of the statistical analysis in wide-tabular
format.</p>
<ul>
<li>
<u>“comparison”;signal_log2_ratio:</u> signal log2-ratios on peptide
basis of the specific comparison</li>
<li>
<u>“comparison”;raw_p_value:</u> raw p-value of the specific
comparison</li>
<li>
<u>“comparison”;adjusted_p_value:</u> adjusted p-value of the
specific comparison</li>
</ul>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:right;">
B_manual/A_manual&amp;#124;signal_log2_ratio
</th>
<th style="text-align:right;">
B_manual/A_manual&amp;#124;raw_p_value
</th>
<th style="text-align:right;">
B_manual/A_manual&amp;#124;adjusted_p_value
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:right;">
-0.0473512
</td>
<td style="text-align:right;">
0.7433455
</td>
<td style="text-align:right;">
0.9010067
</td>
</tr>
<tr>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:right;">
0.0443521
</td>
<td style="text-align:right;">
0.8641528
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:right;">
-0.1165372
</td>
<td style="text-align:right;">
0.1911809
</td>
<td style="text-align:right;">
0.2713360
</td>
</tr>
<tr>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:right;">
0.0066561
</td>
<td style="text-align:right;">
0.9999290
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:right;">
0.0079990
</td>
<td style="text-align:right;">
0.9999709
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:right;">
-0.0022888
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:right;">
-0.1181271
</td>
<td style="text-align:right;">
0.5998730
</td>
<td style="text-align:right;">
0.7532240
</td>
</tr>
<tr>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:right;">
-0.9757784
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:right;">
-0.0529473
</td>
<td style="text-align:right;">
0.2831211
</td>
<td style="text-align:right;">
0.3925563
</td>
</tr>
<tr>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:right;">
-0.0104878
</td>
<td style="text-align:right;">
0.9892582
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O14657
</td>
<td style="text-align:right;">
-0.0820429
</td>
<td style="text-align:right;">
0.1787867
</td>
<td style="text-align:right;">
0.2559203
</td>
</tr>
<tr>
<td style="text-align:left;">
O14818
</td>
<td style="text-align:right;">
-0.0063245
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
</tr>
<tr>
<td style="text-align:left;">
O14893
</td>
<td style="text-align:right;">
-0.0442411
</td>
<td style="text-align:right;">
0.4196139
</td>
<td style="text-align:right;">
0.5551758
</td>
</tr>
<tr>
<td style="text-align:left;">
O14929
</td>
<td style="text-align:right;">
0.0491310
</td>
<td style="text-align:right;">
0.7406270
</td>
<td style="text-align:right;">
0.8992711
</td>
</tr>
<tr>
<td style="text-align:left;">
O14949
</td>
<td style="text-align:right;">
0.0534097
</td>
<td style="text-align:right;">
0.2317136
</td>
<td style="text-align:right;">
0.3251779
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="section level3">
<h3 id="protein-intensity-estimation-comparison">protein intensity estimation comparison<a class="anchor" aria-label="anchor" href="#protein-intensity-estimation-comparison"></a>
</h3>
<p>Since bottom-up proteomics measures ions/peptides and not directly
proteins the protein intensity estimation is slightly biased depending
on the algorithm used. The ROPECA statistics used peptide ratios which
can be compared to protein intensity ratios. This ratio comparison give
researchers insights into the performance and agreement between various
protein intensity estimation algorithms used in their proteomics data
analysis. This information can be valuable for selecting the most
appropriate estimation method and protein candidates per comparison.</p>
<div class="section level4">
<h4 class="tabset tabset-pills" id="processed-data---figures">processed data - figures<a class="anchor" aria-label="anchor" href="#processed-data---figures"></a>
</h4>
<div class="section level5">
<h5 id="protein_intensity_benchmark__scatter_plot">Protein_intensity_benchmark__scatter_plot<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__scatter_plot"></a>
</h5>
<p>The <strong>Protein_intensity_benchmark__scatter_plot</strong> plot
depicts the peptide intensity ratio on the x-axis and the protein
intensity ratio on the y-axis for proteins with 2 or more peptides.</p>
<p>The solid line shows the diagonal and the dashed lines indicate a
2-fold difference. The count highlights the number of proteins which are
affected by a difference above abs. 2 fold-change. The size of the dots
illustrate the number of peptides. Usually proteins with low number of
peptides are stronger affected by a difference between protein intensity
estimation ratios, e.g. MaxLFQ ratios, vs. peptide ratios.</p>
<p><img src="figures/04a__Protein_intensity_benchmark__scatter_plot.png" style="width:60.0%"></p>
</div>
<div class="section level5">
<h5 id="protein_intensity_benchmark__ma_like_plot">Protein_intensity_benchmark__MA_like_plot<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__ma_like_plot"></a>
</h5>
<p>The <strong>Protein_intensity_benchmark__MA_like_plot</strong> plot
depicts an MA-like plot of the mean protein intensity (x-axis) and the
peptide ratio / protein ratio on the y-axis. Usually proteins in the low
abundant range are stronger affected by a difference between protein
intensity estimation ratios and peptide ratios.</p>
<p>The solid line shows the diagonal and the dashed lines indicate a
2fold difference. The count highlights the number of proteins which are
affected by a difference above abs. 2 fold-change. The size of the dots
illustrate the number of peptides. Usually proteins with low number of
peptides are stronger affected by a difference between protein intensity
estimation ratios, e.g. MaxLFQ ratios, and peptide ratios.</p>
<p><img src="figures/04a__Protein_intensity_benchmark__MA_like_plot.png" style="width:80.0%"></p>
</div>
<div class="section level5">
<h5 id="protein_intensity_benchmark__histogram_plot">Protein_intensity_benchmark__histogram_plot<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__histogram_plot"></a>
</h5>
<p>The <strong>Protein_intensity_benchmark__histogram_plot</strong> plot
depicts a histogram of the the peptide ratio / protein ratio on the
x-axis.</p>
<p><img src="figures/04a__Protein_intensity_benchmark__histogram_plot.png" style="width:60.0%"></p>
</div>
<div class="section level5">
<h5 id="protein_intensity_benchmark__fc_gradient_area_plot">Protein_intensity_benchmark__FC_gradient_area_plot<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__fc_gradient_area_plot"></a>
</h5>
<p>The
<strong>Protein_intensity_benchmark__FC_gradient_area_plot</strong>
presents an area plot across a range of fold-changes. In this instance,
at a 1.5-fold change threshold, there are 76 proteins where the protein
ratio is more than 1.5-fold to high, and 2 proteins where the protein
ratio is more than 1.5-fold to low when compared to the peptide
ratios.</p>
<p><img src="figures/04a__Protein_intensity_benchmark__FC_gradient_area_plot.png" style="width:60.0%"></p>
</div>
<div class="section level5">
<h5 id="protein_intensity_benchmark__barplot">Protein_intensity_benchmark__barplot<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__barplot"></a>
</h5>
<p>The <strong>Protein_intensity_benchmark__barplot</strong> displays a
stacked bar plot, summarizing the number of proteins that exhibit a
protein intensity ratio exceeding a 2-fold difference when compared to
the peptide ratio.</p>
<p><img src="figures/04a__Protein_intensity_benchmark__barplot.png" style="width:60.0%"></p>
</div>
</div>
<div class="section level4">
<h4 class="tabset tabset-pills" id="processed-data---tables">processed data - tables<a class="anchor" aria-label="anchor" href="#processed-data---tables"></a>
</h4>
<div class="section level5">
<h5 id="protein_intensity_benchmark__table-csv">Protein_intensity_benchmark__table.csv<a class="anchor" aria-label="anchor" href="#protein_intensity_benchmark__table-csv"></a>
</h5>
<p>The <strong>Protein_intensity_benchmark__table.csv</strong> table
holds all information for the protein intensity ratio vs. peptide
intensity ratios on protein level.</p>
<div class="section level6">
<h6 id="annotation">annotation<a class="anchor" aria-label="anchor" href="#annotation"></a>
</h6>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
parameter
</th>
<th style="text-align:left;">
description
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
slr
</td>
<td style="text-align:left;">
signal log2-ratios on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
t
</td>
<td style="text-align:left;">
t of t-statistics on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
score
</td>
<td style="text-align:left;">
score of t-statistics on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
n
</td>
<td style="text-align:left;">
number of peptides
</td>
</tr>
<tr>
<td style="text-align:left;">
p
</td>
<td style="text-align:left;">
raw p-value of statistics on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
p.fdr
</td>
<td style="text-align:left;">
adjusted p-value (q-value) of statistics on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
PG.ProteinGroups
</td>
<td style="text-align:left;">
Protein groups
</td>
</tr>
<tr>
<td style="text-align:left;">
group1
</td>
<td style="text-align:left;">
group1 of condition comparison
</td>
</tr>
<tr>
<td style="text-align:left;">
group2
</td>
<td style="text-align:left;">
group2 of condition comparison
</td>
</tr>
<tr>
<td style="text-align:left;">
slr_ratio_meta
</td>
<td style="text-align:left;">
how the ratio is formed
</td>
</tr>
<tr>
<td style="text-align:left;">
test
</td>
<td style="text-align:left;">
which test was used for statistics on peptide level
</td>
</tr>
<tr>
<td style="text-align:left;">
type
</td>
<td style="text-align:left;">
which type of ratio aggregation to ProteinGroup level was used for
signal log2-ratios on peptide basis
</td>
</tr>
<tr>
<td style="text-align:left;">
significant_changed
</td>
<td style="text-align:left;">
if there is a significant change FC &amp; q-value (cutoffs e.g.: FC =
1.5 &amp; adjusted-p-value = 0.05)
</td>
</tr>
<tr>
<td style="text-align:left;">
significant_changed_raw_p
</td>
<td style="text-align:left;">
if there is a significant change FC &amp; p-value(cutoffs e.g.: FC = 1.5
&amp; p-value = 0.05)
</td>
</tr>
<tr>
<td style="text-align:left;">
significant_changed_fc
</td>
<td style="text-align:left;">
fold-change cutoff used for analysis
</td>
</tr>
<tr>
<td style="text-align:left;">
significant_changed_p_value
</td>
<td style="text-align:left;">
p-value/q-value cutoff used for analysis
</td>
</tr>
<tr>
<td style="text-align:left;">
fold_change_absolute
</td>
<td style="text-align:left;">
ablsolute fold-change
</td>
</tr>
<tr>
<td style="text-align:left;">
fold_change_direction
</td>
<td style="text-align:left;">
fold-change direction
</td>
</tr>
<tr>
<td style="text-align:left;">
fold_change
</td>
<td style="text-align:left;">
fold-change
</td>
</tr>
<tr>
<td style="text-align:left;">
median_protein_data_ratio
</td>
<td style="text-align:left;">
ratio of protein data (median protein intensity per condtion; ratio
between 2 condtion medians)
</td>
</tr>
<tr>
<td style="text-align:left;">
log2_median_protein_data_ratio
</td>
<td style="text-align:left;">
log2 ratio of protein data (median protein intensity per condtion; ratio
between 2 condtion medians)
</td>
</tr>
<tr>
<td style="text-align:left;">
log2_protein_ratios__peptide_ratios
</td>
<td style="text-align:left;">
log2 RATIOprotein/RATIOpeptide per comparison and ProteinGroup
</td>
</tr>
<tr>
<td style="text-align:left;">
protein_estimation_comment
</td>
<td style="text-align:left;">
is the protein intensity estimate ratio to high or to low in comparsion
to the peptide ratio
</td>
</tr>
<tr>
<td style="text-align:left;">
group1_median_protein_intensity
</td>
<td style="text-align:left;">
median protein intensity estimate of group 1
</td>
</tr>
<tr>
<td style="text-align:left;">
group2_median_protein_intensity
</td>
<td style="text-align:left;">
median protein intensity estimate of group 2
</td>
</tr>
<tr>
<td style="text-align:left;">
group1_median_S2N
</td>
<td style="text-align:left;">
group1 median over signal-to-noise of ions
</td>
</tr>
<tr>
<td style="text-align:left;">
group2_median_S2N
</td>
<td style="text-align:left;">
group2 median over signal-to-noise of ions
</td>
</tr>
<tr>
<td style="text-align:left;">
group1_present_replicate_percentage
</td>
<td style="text-align:left;">
group1 present with selected q-value cutoff with at least 2 peptides per
replicate: percentage of replicates per condition; NA = only 1 peptide
per replicate &gt;&gt; may have globally 2 or more peptide
</td>
</tr>
<tr>
<td style="text-align:left;">
group2_present_replicate_percentage
</td>
<td style="text-align:left;">
group2 present with selected q-value cutoff with at least 2 peptides per
replicate: percentage of replicates per condition; NA = only 1 peptide
per replicate &gt;&gt; may have globally 2 or more peptide
</td>
</tr>
<tr>
<td style="text-align:left;">
direction_of_protein_ratios_and_peptide_ratios
</td>
<td style="text-align:left;">
does the protein ratio and peptide ratio point into the same direction?
</td>
</tr>
<tr>
<td style="text-align:left;">
effect_size_method
</td>
<td style="text-align:left;">
effect size estimation method used (mean scaled peptides intensities are
used as input)
</td>
</tr>
<tr>
<td style="text-align:left;">
d
</td>
<td style="text-align:left;">
effect size estimate
</td>
</tr>
<tr>
<td style="text-align:left;">
d_pooled_SD
</td>
<td style="text-align:left;">
within-groups standard diviation
</td>
</tr>
<tr>
<td style="text-align:left;">
d_95CI_lower
</td>
<td style="text-align:left;">
the lower 95% confidence interval
</td>
</tr>
<tr>
<td style="text-align:left;">
d_95CI_upper
</td>
<td style="text-align:left;">
the upper 95% confidence interval
</td>
</tr>
<tr>
<td style="text-align:left;">
d_magnitute
</td>
<td style="text-align:left;">
a qualitative assessment of the magnitude of effect size
(&amp;#124;d&amp;#124;&lt;0.2 negligible, &amp;#124;d&amp;#124;&lt;0.5
small, &amp;#124;d&amp;#124;&lt;0.8 medium, otherwise large); Cohen 1992
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level6">
<h6 id="table">table<a class="anchor" aria-label="anchor" href="#table"></a>
</h6>
<table class="table table" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:right;">
slr
</th>
<th style="text-align:right;">
t
</th>
<th style="text-align:right;">
score
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
p
</th>
<th style="text-align:right;">
p.fdr
</th>
<th style="text-align:left;">
PG.ProteinGroups
</th>
<th style="text-align:left;">
group1
</th>
<th style="text-align:left;">
group2
</th>
<th style="text-align:left;">
slr_ratio_meta
</th>
<th style="text-align:left;">
test
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:left;">
significant_changed
</th>
<th style="text-align:left;">
significant_changed_raw_p
</th>
<th style="text-align:right;">
significant_changed_fc
</th>
<th style="text-align:right;">
significant_changed_p_value
</th>
<th style="text-align:right;">
fold_change_absolute
</th>
<th style="text-align:left;">
fold_change_direction
</th>
<th style="text-align:right;">
fold_change
</th>
<th style="text-align:left;">
effect_size_method
</th>
<th style="text-align:right;">
d
</th>
<th style="text-align:right;">
d_pooled_SD
</th>
<th style="text-align:right;">
d_95CI_lower
</th>
<th style="text-align:right;">
d_95CI_upper
</th>
<th style="text-align:left;">
d_magnitute
</th>
<th style="text-align:right;">
median_protein_data_ratio
</th>
<th style="text-align:right;">
log2_median_protein_data_ratio
</th>
<th style="text-align:right;">
log2_protein_ratios__peptide_ratios
</th>
<th style="text-align:left;">
protein_estimation_comment
</th>
<th style="text-align:right;">
group1_median_protein_intensity
</th>
<th style="text-align:right;">
group2_median_protein_intensity
</th>
<th style="text-align:right;">
group1_median_S2N
</th>
<th style="text-align:right;">
group2_median_S2N
</th>
<th style="text-align:right;">
group1_present_replicate_percentage
</th>
<th style="text-align:right;">
group2_present_replicate_percentage
</th>
<th style="text-align:left;">
direction_of_protein_ratios_and_peptide_ratios
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:right;">
-0.0473512
</td>
<td style="text-align:right;">
-0.4068343
</td>
<td style="text-align:right;">
0.6962885
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.7433455
</td>
<td style="text-align:right;">
0.9010067
</td>
<td style="text-align:left;">
A0PJW6
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.033366
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.033366
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2791355
</td>
<td style="text-align:right;">
0.1074963
</td>
<td style="text-align:right;">
-1.3567385
</td>
<td style="text-align:right;">
0.7984675
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
0.8362415
</td>
<td style="text-align:right;">
-0.2580085
</td>
<td style="text-align:right;">
-0.2106574
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
13769.343
</td>
<td style="text-align:right;">
16465.750
</td>
<td style="text-align:right;">
11.862334
</td>
<td style="text-align:right;">
12.819728
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0443521
</td>
<td style="text-align:right;">
0.4408390
</td>
<td style="text-align:right;">
0.6726308
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.8641528
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
A1X283
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.031220
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0164415
</td>
<td style="text-align:right;">
0.1860178
</td>
<td style="text-align:right;">
-0.4536598
</td>
<td style="text-align:right;">
0.4865428
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.9013207
</td>
<td style="text-align:right;">
-0.1498876
</td>
<td style="text-align:right;">
-0.1942397
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
5831.712
</td>
<td style="text-align:right;">
6470.186
</td>
<td style="text-align:right;">
4.011297
</td>
<td style="text-align:right;">
5.433860
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
contrary directions
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1165372
</td>
<td style="text-align:right;">
-1.0692391
</td>
<td style="text-align:right;">
0.3204605
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
0.1911809
</td>
<td style="text-align:right;">
0.2713360
</td>
<td style="text-align:left;">
L0R6Q1
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.084130
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.084130
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6380376
</td>
<td style="text-align:right;">
0.1449498
</td>
<td style="text-align:right;">
-1.2942932
</td>
<td style="text-align:right;">
0.0182179
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
0.7927020
</td>
<td style="text-align:right;">
-0.3351495
</td>
<td style="text-align:right;">
-0.2186123
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
22810.444
</td>
<td style="text-align:right;">
28775.561
</td>
<td style="text-align:right;">
11.717550
</td>
<td style="text-align:right;">
8.904539
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0066561
</td>
<td style="text-align:right;">
0.1008351
</td>
<td style="text-align:right;">
0.9225103
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0.9999290
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00330
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.004624
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1792522
</td>
<td style="text-align:right;">
0.1897052
</td>
<td style="text-align:right;">
-0.2454271
</td>
<td style="text-align:right;">
0.6039315
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8330499
</td>
<td style="text-align:right;">
-0.2635252
</td>
<td style="text-align:right;">
-0.2701814
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
8532.935
</td>
<td style="text-align:right;">
10243.006
</td>
<td style="text-align:right;">
8.159923
</td>
<td style="text-align:right;">
9.367225
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
contrary directions
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0079990
</td>
<td style="text-align:right;">
-0.0748142
</td>
<td style="text-align:right;">
0.9424568
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9999709
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00487
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.005560
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.0573662
</td>
<td style="text-align:right;">
0.3145541
</td>
<td style="text-align:right;">
-0.3878923
</td>
<td style="text-align:right;">
0.5026247
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8300690
</td>
<td style="text-align:right;">
-0.2686968
</td>
<td style="text-align:right;">
-0.2766957
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
21651.349
</td>
<td style="text-align:right;">
26083.793
</td>
<td style="text-align:right;">
9.446892
</td>
<td style="text-align:right;">
11.970736
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
contrary directions
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0022888
</td>
<td style="text-align:right;">
-0.0321861
</td>
<td style="text-align:right;">
0.9752227
</td>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O00571
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.001588
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.001588
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.0370422
</td>
<td style="text-align:right;">
0.1975283
</td>
<td style="text-align:right;">
-0.2794375
</td>
<td style="text-align:right;">
0.2053530
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8616703
</td>
<td style="text-align:right;">
-0.2147921
</td>
<td style="text-align:right;">
-0.2125032
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
53132.494
</td>
<td style="text-align:right;">
61662.205
</td>
<td style="text-align:right;">
12.812127
</td>
<td style="text-align:right;">
13.289387
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.1181271
</td>
<td style="text-align:right;">
-0.6426256
</td>
<td style="text-align:right;">
0.5409473
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.5998730
</td>
<td style="text-align:right;">
0.7532240
</td>
<td style="text-align:left;">
O00622
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.085325
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.085325
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1723566
</td>
<td style="text-align:right;">
0.3216608
</td>
<td style="text-align:right;">
-0.2986087
</td>
<td style="text-align:right;">
0.6433219
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8364571
</td>
<td style="text-align:right;">
-0.2576366
</td>
<td style="text-align:right;">
-0.1395095
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
5735.594
</td>
<td style="text-align:right;">
6857.009
</td>
<td style="text-align:right;">
3.709370
</td>
<td style="text-align:right;">
4.977050
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.9757784
</td>
<td style="text-align:right;">
-8.2918171
</td>
<td style="text-align:right;">
0.0000727
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:left;">
O13516
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.966702
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.966702
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-2.1848927
</td>
<td style="text-align:right;">
0.3573362
</td>
<td style="text-align:right;">
-2.7789090
</td>
<td style="text-align:right;">
-1.5908763
</td>
<td style="text-align:left;">
large
</td>
<td style="text-align:right;">
0.4166547
</td>
<td style="text-align:right;">
-1.2630758
</td>
<td style="text-align:right;">
-0.2872973
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
17678.528
</td>
<td style="text-align:right;">
42429.683
</td>
<td style="text-align:right;">
7.777781
</td>
<td style="text-align:right;">
14.886838
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0529473
</td>
<td style="text-align:right;">
-0.9991841
</td>
<td style="text-align:right;">
0.3510038
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0.2831211
</td>
<td style="text-align:right;">
0.3925563
</td>
<td style="text-align:left;">
O14561
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.037382
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.037382
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.6291879
</td>
<td style="text-align:right;">
0.2031349
</td>
<td style="text-align:right;">
-1.4965383
</td>
<td style="text-align:right;">
0.2381626
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
0.7971930
</td>
<td style="text-align:right;">
-0.3269991
</td>
<td style="text-align:right;">
-0.2740518
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
15594.825
</td>
<td style="text-align:right;">
19562.170
</td>
<td style="text-align:right;">
10.766962
</td>
<td style="text-align:right;">
10.420646
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0104878
</td>
<td style="text-align:right;">
-0.2441598
</td>
<td style="text-align:right;">
0.8141130
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
0.9892582
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14579
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.007296
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.007296
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.4053804
</td>
<td style="text-align:right;">
0.1706905
</td>
<td style="text-align:right;">
-0.0443356
</td>
<td style="text-align:right;">
0.8550963
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
0.8398016
</td>
<td style="text-align:right;">
-0.2518795
</td>
<td style="text-align:right;">
-0.2413917
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
38715.729
</td>
<td style="text-align:right;">
46101.042
</td>
<td style="text-align:right;">
11.925857
</td>
<td style="text-align:right;">
13.447613
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0820429
</td>
<td style="text-align:right;">
-1.0571006
</td>
<td style="text-align:right;">
0.3255960
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
0.1787867
</td>
<td style="text-align:right;">
0.2559203
</td>
<td style="text-align:left;">
O14657
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.058516
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.058516
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.1920303
</td>
<td style="text-align:right;">
0.2575590
</td>
<td style="text-align:right;">
-0.3903803
</td>
<td style="text-align:right;">
0.7744409
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8233938
</td>
<td style="text-align:right;">
-0.2803455
</td>
<td style="text-align:right;">
-0.1983026
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
10757.665
</td>
<td style="text-align:right;">
13065.031
</td>
<td style="text-align:right;">
4.936353
</td>
<td style="text-align:right;">
5.980577
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0063245
</td>
<td style="text-align:right;">
-0.0259550
</td>
<td style="text-align:right;">
0.9800181
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:right;">
1.0000000
</td>
<td style="text-align:left;">
O14818
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.004393
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.004393
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.0140356
</td>
<td style="text-align:right;">
0.2473297
</td>
<td style="text-align:right;">
-0.3885587
</td>
<td style="text-align:right;">
0.3604874
</td>
<td style="text-align:left;">
negligible
</td>
<td style="text-align:right;">
0.8519391
</td>
<td style="text-align:right;">
-0.2311779
</td>
<td style="text-align:right;">
-0.2248534
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
40482.118
</td>
<td style="text-align:right;">
47517.622
</td>
<td style="text-align:right;">
17.675102
</td>
<td style="text-align:right;">
17.970990
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
-0.0442411
</td>
<td style="text-align:right;">
-0.8248106
</td>
<td style="text-align:right;">
0.4366954
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.4196139
</td>
<td style="text-align:right;">
0.5551758
</td>
<td style="text-align:left;">
O14893
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.031141
</td>
<td style="text-align:left;">
down
</td>
<td style="text-align:right;">
-1.031141
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
-0.2370409
</td>
<td style="text-align:right;">
0.1437852
</td>
<td style="text-align:right;">
-1.3131936
</td>
<td style="text-align:right;">
0.8391119
</td>
<td style="text-align:left;">
small
</td>
<td style="text-align:right;">
0.8145117
</td>
<td style="text-align:right;">
-0.2959927
</td>
<td style="text-align:right;">
-0.2517516
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
8520.376
</td>
<td style="text-align:right;">
10460.717
</td>
<td style="text-align:right;">
6.269485
</td>
<td style="text-align:right;">
6.261318
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
same direction - down
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0491310
</td>
<td style="text-align:right;">
0.5638555
</td>
<td style="text-align:right;">
0.5904612
</td>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
0.7406270
</td>
<td style="text-align:right;">
0.8992711
</td>
<td style="text-align:left;">
O14929
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.034641
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.034641
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.6283915
</td>
<td style="text-align:right;">
0.1706648
</td>
<td style="text-align:right;">
0.2132161
</td>
<td style="text-align:right;">
1.0435668
</td>
<td style="text-align:left;">
medium
</td>
<td style="text-align:right;">
0.8979881
</td>
<td style="text-align:right;">
-0.1552317
</td>
<td style="text-align:right;">
-0.2043627
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
19027.330
</td>
<td style="text-align:right;">
21188.843
</td>
<td style="text-align:right;">
10.134418
</td>
<td style="text-align:right;">
8.927515
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
contrary directions
</td>
</tr>
<tr>
<td style="text-align:right;">
0.0534097
</td>
<td style="text-align:right;">
1.1650318
</td>
<td style="text-align:right;">
0.2821887
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
0.2317136
</td>
<td style="text-align:right;">
0.3251779
</td>
<td style="text-align:left;">
O14949
</td>
<td style="text-align:left;">
B_manual
</td>
<td style="text-align:left;">
A_manual
</td>
<td style="text-align:left;">
B_manual/A_manual
</td>
<td style="text-align:left;">
modt
</td>
<td style="text-align:left;">
median
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:left;">
none
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.05
</td>
<td style="text-align:right;">
1.037715
</td>
<td style="text-align:left;">
up
</td>
<td style="text-align:right;">
1.037715
</td>
<td style="text-align:left;">
Cohen’s d
</td>
<td style="text-align:right;">
0.9962874
</td>
<td style="text-align:right;">
0.0374448
</td>
<td style="text-align:right;">
-0.1406891
</td>
<td style="text-align:right;">
2.1332639
</td>
<td style="text-align:left;">
large
</td>
<td style="text-align:right;">
0.8800108
</td>
<td style="text-align:right;">
-0.1844069
</td>
<td style="text-align:right;">
-0.2378166
</td>
<td style="text-align:left;">
OK
</td>
<td style="text-align:right;">
72653.469
</td>
<td style="text-align:right;">
82559.746
</td>
<td style="text-align:right;">
37.760534
</td>
<td style="text-align:right;">
50.907383
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
contrary directions
</td>
</tr>
</tbody>
</table>
<p>The <strong>Protein_intensity_benchmark__table_top15.csv</strong>
table holds the information of the Top15 most deviating ratio / ratio
comparisons.</p>
</div>
</div>
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Stephan Michalik.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
